<!DOCTYPE html>
<!--[if lt IE 7]>      <html class="no-js lt-ie9 lt-ie8 lt-ie7"> <![endif]-->
<!--[if IE 7]>         <html class="no-js lt-ie9 lt-ie8"> <![endif]-->
<!--[if IE 8]>         <html class="no-js lt-ie9"> <![endif]-->
<!--[if gt IE 8]><!--> 
<html> <!--<![endif]-->
<head>
  <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
  <meta name="description" content="RECOMB 2024 - Bilkent University, Ankara">
  
  <meta name="author" content="Themefisher.com">

  <title>RECOMB 2024 | PROGRAM</title>

  <link rel="alternate" href="http://recomb2024.bilkent.edu.tr" hreflang="en-gb" />
  <link rel="alternate" href="http://recomb2024.bilkent.edu.tr" hreflang="en-us" />

<!-- Mobile Specific Meta
  ================================================== -->
  <meta name="viewport" content="width=device-width, initial-scale=1">
  
  <!-- Favicon -->
  <link rel="shortcut icon" type="image/x-icon" href="images/favicon.png" />
  
  <!-- CSS
  ================================================== -->
  <!-- Themefisher Icon font -->
  <link rel="stylesheet" href="plugins/themefisher-font/style.css">
  <!-- bootstrap.min css -->
  <link rel="stylesheet" href="plugins/bootstrap/css/bootstrap.min.css">
  <!-- Lightbox.min css -->
  <link rel="stylesheet" href="plugins/lightbox2/dist/css/lightbox.min.css">
  <!-- animation css -->
  <link rel="stylesheet" href="plugins/animate/animate.css">
  <!-- Slick Carousel -->
  <link rel="stylesheet" href="plugins/slick/slick.css">
  <!-- Main Stylesheet -->
  <link rel="stylesheet" href="css/style.css">  
  
  <script src="https://kit.fontawesome.com/5ce389c680.js" crossorigin="anonymous"></script>

</head>

<body id="body">

 <!-- Start Preloader
  ==================================== -->
  <div id="preloader">
    <div class='preloader'>
      <span></span>
      <span></span>
      <span></span>
      <span></span>
      <span></span>
      <span></span>
    </div>
  </div> 
<!-- End Preloader
  ==================================== -->

<!-- Fixed Navigation
  ==================================== -->
<header class="navigation fixed-top">
  <div class="container">
    <!-- main nav -->
    
    <nav class="navbar navbar-expand-lg navbar-light">
      <!-- logo -->
      <a class="navbar-brand logo" href="index.html">
        <img class="logo-default" src="images/recomb-logo/logo-name.png" alt="RECOMB Conference Logo"/>
        <img class="logo-white" src="images/recomb-logo/logo-name-white.png" alt="RECOMB Conference Logo"/>
      </a>
      <!-- /logo -->
      <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navigation"
        aria-controls="navigation" aria-expanded="false" aria-label="Toggle navigation">
        <span class="navbar-toggler-icon"></span>
      </button>

      <div class="collapse navbar-collapse" id="navigation">
        <ul class="navbar-nav mr-auto text-center">
          <li class="nav-item ">
            <a class="nav-link" href="index.html">Home</a>
          </li>
          <li class="nav-item dropdown active">
            <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown"
              aria-haspopup="true" aria-expanded="false">Conference</a>
            <div class="dropdown-menu" aria-labelledby="navbarDropdown">
              <a class="dropdown-item" href="committees.html">Committees</a>
              <a class="dropdown-item" href="keydates_and_deadlines.html">Key dates & Deadlines</a>
              <a class="dropdown-item" href="call_for_papers.html">Call for Papers</a>
              <a class="dropdown-item" href="call_for_abstracts.html">Call for Abstracts</a>
              <a class="dropdown-item" href="call_for_highlights.html">Call for Highlights</a>
              <!-- <a class="dropdown-item" href="program.html">Program</a> -->
              <!-- <a class="dropdown-item" href="posters.html">Posters</a> -->
            </div>
          </li>
          <li class="nav-item dropdown active">
            <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown"
              aria-haspopup="true" aria-expanded="false">Register</a>
            <div class="dropdown-menu" aria-labelledby="navbarDropdown">
              <a class="dropdown-item" href="register.html">RECOMB and Satellites</a>
              <a class="dropdown-item" href="travel_fellowships.html">Travel Fellowships</a>
              <!-- <a class="dropdown-item" href="accepted_papers.html">Accepted Papers</a> -->
              <!-- <a class="dropdown-item" href="posters.html">Posters</a> -->
            </div>
          </li>
          <li class="nav-item dropdown active">
            <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown"
              aria-haspopup="true" aria-expanded="false">Program</a>
            <div class="dropdown-menu" aria-labelledby="navbarDropdown">
              <a class="dropdown-item" href="program.html">Program Schedule</a>
              <a class="dropdown-item" href="keynotes.html">Keynotes</a>
              <a class="dropdown-item" href="posters.html">Poster Guidelines</a>
              <!-- <a class="dropdown-item" href="accepted_papers.html">Accepted Papers</a> -->
              <!-- <a class="dropdown-item" href="call_for_abstracts.html#submission">Travel Fellowship</a> -->
              <!-- <a class="dropdown-item" href="posters.html">Posters</a> -->
            </div>
          </li>
          <li class="nav-item dropdown active">
            <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown"
              aria-haspopup="true" aria-expanded="false">Venue</a>
             <div class="dropdown-menu" aria-labelledby="navbarDropdown">
              <a class="dropdown-item" href="venue_main.html">Main Conference Venue</a>
              <a class="dropdown-item" href="venue_satellites.html">Satellites Venue</a>
              <a class="dropdown-item" href="hotels.html">Hotels</a>
              <a class="dropdown-item" href="airport.html">Airport Directions</a>
	      <a class="dropdown-item" href="things_to_do.html">Things to do @Cambridge</a>
              <a class="dropdown-item" href="transportation.html">Local Transportation</a>
	      <a class="dropdown-item" href="visa.html">Visa Information</a>
            </div>
          </li>
          <li class="nav-item dropdown active">
            <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown"
              aria-haspopup="true" aria-expanded="false">Satellites</a>
             <div class="dropdown-menu" aria-labelledby="navbarDropdown">
              <a class="dropdown-item" href="https://recomb-seq.github.io/" target="_blank">RECOMB-Seq</a>
              <a class="dropdown-item" href="https://recomb.org/recomb-ccb2024" target="_blank">RECOMB-CCB</a>
              <a class="dropdown-item" href="https://recomb-cg.org/" target="_blank">RECOMB-CG</a>
              <a class="dropdown-item" href="recombgenetics.html">RECOMB-Genetics</a>
            </div>
          </li>
          <li class="nav-item dropdown active">
            <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown"
              aria-haspopup="true" aria-expanded="false">Sponsors</a>
             <div class="dropdown-menu" aria-labelledby="navbarDropdown">
              <a class="dropdown-item" href="sponsors.html">RECOMB Sponsors</a>
              <a class="dropdown-item" href="sponsorship_packages.html">Sponsorship Packages</a>
            </div>
          </li>
          <li class="nav-item ">
            <a href="https://www.recomb.org" target="blank" style="color:white;">About RECOMB</a>
          </li>
        </ul>
      </div>
    </nav>
    <!-- /main nav -->
  </div>
</header>
<!-- End Fixed Navigation
  ==================================== -->


<section class="single-page-header">
	<div class="container">
		<div class="row">
			<div class="col-md-12">
				<h2>Program</h2>
			</div>
		</div>
	</div>
</section>


<!-- Start Program
    ==================================== -->
<section class="team-skills section-sm" id="skills">
  <div class="container">
    <div class="col-12">
      <div class="team-skills-content center">

        <hr> <h2>Accepted Papers</h2> <hr>
      </div>

<ul>
<li> Ghanshyam Chandra and Chirag Jain. "Sequence to graph alignment using gap-sensitive co-linear chaining".</li>
<li>Nathan LaPierre, Boyang Fu, Steven Turnbull, Eleazar Eskin and Sriram Sankararaman. "Leveraging family data to design Mendelian Randomization that is provably robust to population stratification". </li>
<li>Yasamin Tabatabaee, Sebastien Roch and Tandy Warnow. "Statistically Consistent Rooting of Species Trees under the Multi-Species Coalescent Model". </li>
<li>Jiacheng Lin, Hanwen Xu, Addie Woicik, Jianzhu Ma and Sheng Wang. "Pisces: A combo-wise contrastive learning approach to synergistic drug combination prediction". </li>
<li>Uyen Mai, Eduardo Charvel and Siavash Mirarab. "MD-Cat: Phylogenetic dating under a flexible categorical model using Expectation-Maximization". </li>
<li>Louxin Zhang, Niloufar Abhari, Caroline Colijn and Yufeng Wu. "A Fast and Scalable Method for Inferring Phylogenetic Networks from Trees via Aligning the Ancestor Sequences of Taxa". </li>
<li>Xiaogen Zhou, Zhiqiang Li and Tong Tong. "DM-Net: A Dual-Model Network for Automated Biomedical Image Diagnosis". </li>
<li>Jim Shaw and Yun William Yu. "Seed-chain-extend alignment is accurate and runs in close to O(m log n) time for similar sequences: a rigorous average-case analysis". </li>
<li>Soufiane M.C. Mourragui, Marco Loog, Mirrelijn van Nee, Mark van de Wiel, Marcel J.T. Reinders and Lodewyk F.A. Wessels. "Percolate: an exponential family JIVE model to design DNA-based predictors of drug response". </li>
<li>Amir Joudaki, Alexandru Meterez, Harun Mustafa, Ragnar Groot Koerkamp, André Kahles and Gunnar Rätsch. "Aligning Distant Sequences to Graphs using Long Seed Sketches". </li>
<li>Daniel Baker and Ben Langmead. "Dashing 2: genomic sketching with multiplicities and locality-sensitive hashing". </li>
<li>Bing-Xue Du, Yi Xu, Siu-Ming Yiu, Hui Yu and Jian-Yu Shi. "MTGL-ADMET: A Novel Multi-Task Graph Learning Framework for ADMET Prediction Enhanced by Status-Theory and Maximum Flow". </li>
<li>Andrea Cracco and Alexandru I. Tomescu. "Extremely-fast construction and querying of compacted and colored de Bruijn graphs with GGCAT". </li>
<li>Sara C. Schulte, Alexander Dilthey and Gunnar W. Klau. "HOGVAX: Exploiting Peptide Overlaps to Maximize Population Coverage in Vaccine Design with Application to SARS-CoV-2". </li>
<li>Benjamin Maier and Kristoffer Sahlin. "Entropy predicts fuzzy-seed sensitivity". </li>
<li>Rıza Özçelik, Alperen Bağ, Berk Atıl, Melih Barsbey, Arzucan Ozgur and Elif Ozkirimli. "DebiasedDTA: Improving the Generalizability of Drug-Target Affinity Prediction Models". </li>
<li>Mahmudur Rahman Hera, N. Tessa Pierce-Ward and David Koslicki. "Deriving confidence intervals for mutation rates across a wide range of evolutionary distances using FracMinHash". </li>
<li>David Pellow, Lianrong Pu, Baris Ekim, Lior Kotlar, Ron Shamir and Yaron Orenstein. "Efficient minimizer orders for large values of k using minimum decycling sets". </li>
<li>Shuvom Sadhuka, Daniel Fridman, Bonnie Berger and Hyunghoon Cho. "Accurate evaluation of transcriptomic re-identification risks using discriminative sequence models". </li>
<li>Jason Fan, Jamshed Khan, Giulio Pibiri and Rob Patro. "Spectrum preserving tilings enable sparse and modular reference indexing". </li>
<li>Omar Ahmed, Massimiliano Rossi, Christina Boucher and Ben Langmead. "Efficient taxa identification using a pangenome index". </li>
<li>Baris Ekim, Kristoffer Sahlin, Paul Medvedev, Bonnie Berger and Rayan Chikhi. "hifimap: Efficient low-divergence mapping of long reads in minimizer space". </li>
<li>Stefan Ivanovic and Mohammed El-Kebir. "Modeling and Predicting Cancer Clonal Evolution with Reinforcement Learning". </li>
<li>Palash Sashittal, Henri Schmidt, Michelle Chan and Benjamin Raphael. "Startle: a star homoplasy approach for CRISPR-Cas9 lineage tracing". </li>
<li>Yudi Zhang, Ha Vu, Karin Dorman and Geetu Tuteja. "Unsupervised Deep Peak Caller for ATAC-seq". </li>
<li>Ahsan Sanaullah, Degui Zhi and Shaojie Zhang. "Minimal Positional Substring Cover: A Haplotype Threading Alternative to Li & Stephens Model". </li>
<li>Yunheng Han and Erin Molloy. "TREE-QMC: Improving quartet graph construction for scalable and accurate species tree estimation from gene trees". </li>
<li>Aditya Gorla, Sriram Sankararaman, Esteban Burchard, Jonathan Flint, Noah Zaitlen and Elior Rahmani. "Phenotypic subtyping via contrastive learning". </li>
<li>Shikha Mallick and Sahely Bhadra. "CDGCN: Conditional de novo Drug generative model using Graph Convolution Networks". </li>
<li>Hao Chen, Dongshunyi Li and Ziv Bar-Joseph. "Cell segmentation for high-resolution spatial transcriptomics".</li>
<li>Regev Schweiger and Richard Durbin. "Ultra-fast genome-wide inference of pairwise coalescence times".</li>
<li>Weizhong Zheng, John H.C. Fong, Yuk Kei Wan, Athena H.Y. Chu, Yuanhua Huang, Alan S.L. Wong and Joshua W.K. Ho. "Translation rate prediction and regulatory motif discovery with multi-task learning".</li>
<li>Runpeng Luo and Yu Lin. "VStrains: De Novo Reconstruction of Viral Strains Via Iterative Path Extraction From Assembly Graphs".</li>
<li>Ziqi Chen, Martin Min, Hongyu Guo, Chao Cheng, Trevor Clancy and Xia Ning. "T-Cell Receptor Optimization with Reinforcement Learning and Mutation Polices for Precision Immunotherapy".</li>
<li>Rajagopal Venkatesaramani, Zhiyu Wan, Bradley Malin and Yevgeniy Vorobeychik. "Enabling Trade-offs in Privacy and Utility in Genomic Data Beacons and Summary Statistics".</li>
<li>Ardalan Naseri, William Yue, Shaojie Zhang and Degui Zhi. "FastRecomb: Fast inference of genetic recombination rates in biobank scale data".</li>
<li>Spencer Krieger and John Kececioglu. "Computing shortest hyperpaths for pathway inference in cellular reaction networks".</li>
<li>Claire McWhite and Mona Singh. "Vector-clustering Multiple Sequence Alignment: Aligning into twilight zone of protein sequence similarity with protein language models".</li>
<li>Rohit Singh, Alexander Wu, Anish Mudide and Bonnie Berger. "Unraveling causal gene regulation from the RNA velocity graph using Velorama".</li>
<li>Xinhao Liu, Ron Zeira and Benjamin J. Raphael. "PASTE2: Partial Alignment of Multi-slice Spatially Resolved Transcriptomics Data".</li>
<li>Yuelin Liu, Xuan C. Li, Farid Rashidi Mehrabadi, Alejandro A. Schäffer, Drew Pratt, David R. Crawford, Salem Malikić, Erin K. Molloy, Vishaka Gopalan, Stephen M. Mount, Eytan Ruppin, Kenneth Aldape and S. Cenk Sahinalp. "Single-cell methylation sequencing data reveals succinct metastatic migration histories and tumor progression models".</li>
<li>Vtalii Stebliankin, Azam Shirali, Prabin Baral, Prem Chapagain and Giri Narasimhan. "PIsToN: Evaluating Protein Binding Interfaces with Transformer Networks".</li>
<li>Chihao Zhang, Yiling Elaine Chen, Shihua Zhang and Jingyi Jessica Li. "Information-theoretic Classification Accuracy: A Criterion that Guides Data-driven Combination of Ambiguous Outcome Labels in Multi-class Classification".</li>
<li>Zhihang Hu, Qinze Yu, Yucheng Guo, Taifeng Wang, Irwin King, Xin Gao, Le Song and Yu Li. "Drug Synergistic Combinations Predictions via Large-Scale Pre-Training and Graph Structure Learning".</li>

</ul> <br />

    </div>    
  </div> <!-- End container -->
</section> <!-- End section -->



    
<section class="program section" id="program">
  <div class="container">
    <div class="row justify-content-center">

      <!-- section title 
      <div class="col-12">
        <div class="title text-center">
          <div class="border"></div>
          <h2>Program</h2> 
          <div class="border"></div>
        </div>
      </div>-->




      <!-- /section title -->

<!--

      <div class="col-md-6 col-sm-12 col-xs-12 col-12">
        
        <h1>Day 1</h1>
        
        <hr>

        <div class="row">
          <div class="col-md-3 col-sm-3 col-xs-3 col-4"><p class="time">09:00 - 09:15</p></div>
          <div class="col-md-9 col-sm-9 col-xs-9 col-8"><p class="activity"><b>Opening Remarks</b></p></div>
        </div>

         <div class="row">
          <div class="col-md-3 col-sm-3 col-xs-3 col-4"><p class="time">09:15 - 09:45</p></div>
          <div class="col-md-5 col-sm-5 col-xs-5 col-6"><p class="invited-speaker">Martin Kircher</p></div>
          <div class="col-md-4 col-sm-4 col-xs-4 col-2">
            <button type="button" class="btn-invited" data-toggle="collapse" data-target="#invited1">&nbsp;&#x2b07;&nbsp; Invited Talk</button>
          </div>
        </div>

        <div class="row">
          <div class="col-md-3 col-sm3 col-xs-3 col-4"></div>
          <div class="col-md-9 col-sm9 col-xs-9 col-8"><div id="invited1" class="collapse">
            <p class="text">
                <b>CADD & CADD-SV – scoring deleteriousness of all genomic variants</b>
                <br><br>
                <button class="btn-abstract" data-toggle="collapse" data-target="#collapsei1"><i class="fas fa-pencil-alt" aria-hidden="true"></i>Abstract</button>
                <div class="collapse" id="collapsei1">
                  <p class="text">
                    Approaches for the identification of disease causal mutations are widely applied in research and clinical settings, but interpretation and ranking of the resulting variants remains challenging. In 2014, we published a metric that objectively weights and integrates collections of annotations. Combined Annotation Dependent Depletion (CADD, https://cadd.gs.washington.edu) integrates annotations by contrasting variants that survived purifying selection along the human lineage with simulated mutations to score short sequence variants (SNVs, InDels, multi-allelic substitutions). CADD was well adopted by the community and minor adjustments and fixes were released since, including the native support of both GRCh37 and GRCh38 assemblies. Recently, we assessed existing deep neural network (DNN) models for splice effects with the Multiplexed Functional Assay of Splicing using Sort-seq dataset (MFASS, Cheung et al. Mol Cell. 2019). We selected two DNN models based only on genomic sequence, MMSplice and SpliceAI, which showed the best performance for integration into CADD. The DNN scores boosted CADD's predictions for splice effects and we noted that while the DNN scores have superior performance on splice variants, they do not compete in general variant effect prediction, as they do not account for nonsense and missense effects of the same variants. This suggests that variant prioritization will improve with more domain-specific information and underlines the importance of identifying additional such features, e.g. for regulatory sequences. With rapid advances in the identification of structural variants (SVs), we decided to apply the general concept of CADD to score them (CADD-SV). While methods utilizing individual mechanistic principles like the deletion of coding sequence or 3D architecture disruptions were available, a comprehensive tool that uses the broad spectrum of available SV annotations was missing. As a proof-of-principle, we show that CADD-SV scores are predictive of pathogenicity and population frequency and that CADD-SV's ability to prioritize pathogenic variants exceeds that of existing methods (0.981 AUROC compared to SVScore 0.967, AnnotSV 0.818). Our results highlight advantages of CADD, like profiting from a large training data set covering diverse and rare feature annotations without major ascertainment effects from historic and on-going variant collections.
                  </p>
                </div>
              </p>
              </div></div>
        </div>

        <div class="row">
          <div class="col-md-3 col-sm-3 col-xs-3 col-4"><p class="time">09:45 - 10:15</p></div>
          <div class="col-md-5 col-sm-5 col-xs-5 col-6"><p class="invited-speaker">Gioele La Manno</p></div>
          <div class="col-md-4 col-sm-4 col-xs-4 col-2">
            <button type="button" class="btn-invited" data-toggle="collapse" data-target="#invited2">&nbsp;&#x2b07;&nbsp; Invited Talk</button>
          </div>
        </div>

        <div class="row">
          <div class="col-md-3 col-sm3 col-xs-3 col-4"></div>
          <div class="col-md-9 col-sm9 col-xs-9 col-8"><div id="invited2" class="collapse">
            <p class="text">
                <b>Molecular architecture of the developing mouse brain</b>
                <br><br>
                <button class="btn-abstract" data-toggle="collapse" data-target="#collapsei2"><i class="fas fa-pencil-alt" aria-hidden="true"></i>Abstract</button>
                <div class="collapse" id="collapsei2">
                  <p class="text">
                    The mammalian brain develops through a complex interplay of spatial cues generated by diffusible morphogens, cell–cell interactions and intrinsic genetic programs that result in probably more than a thousand distinct cell types. A complete understanding of this process requires a systematic characterization of cell states over the entire spatiotemporal range of brain development. The ability of single-cell RNA sequencing and spatial transcriptomics to reveal the molecular heterogeneity of complex tissues has therefore been particularly powerful in the nervous system. Previous studies have explored development in specific brain regions, the whole adult brain9 and even entire embryos10. Here we report a comprehensive single-cell transcriptomic atlas of the embryonic mouse brain between gastrulation and birth. We identified almost eight hundred cellular states that describe a developmental program for the functional elements of the brain and its enclosing membranes, including the early neuroepithelium, region-specific secondary organizers, and both neurogenic and gliogenic progenitors. We also used in situ mRNA sequencing to map the spatial expression patterns of key developmental genes. Integrating the in situ data with our single-cell clusters revealed the precise spatial organization of neural progenitors during the patterning of the nervous system.
                  </p>
                </div>
              </p>
              </div></div>
        </div>

        <div class="row">
          <div class="col-md-3 col-sm-3 col-xs-3 col-4"><p class="time">10:15 - 10:30</p></div>
          <div class="col-md-9 col-sm-9 col-xs-9 col-8"><p class="activity">Break</p></div>
        </div>

        <div class="row">
          
          <div class="col-md-3 col-sm-3 col-xs-3 col-4"><p class="time">10:30 - 11:30</p></div>
          
          <div class="col-md-9 col-sm-9 col-xs-9 col-8">
            
            <button type="button" class="btn" data-toggle="collapse" data-target="#talk1">&nbsp;&nbsp;&nbsp; &#x2b07; Selected Talk Session - 1</button>

            <div id="talk1" class="collapse">

              <p class="text">

                <br>
                <b>AnGenoV: A toolbox for analysis of genomic variants</b>
                <br>
                <i>Sevcan Doğramacı, Mert Doğan, Mehmet Gürsel Arslan, Süleyman Emre Çelik, Tuğba Nur Korkmaz and Arda Söylev</i>
                <br><br>
                <button class="btn-abstract" data-toggle="collapse" data-target="#collapses1"><i class="fas fa-pencil-alt" aria-hidden="true"></i>Abstract</button>
                <div class="collapse" id="collapses1">
                  <p class="text">
                    Evolution of genomes has been conducing to several variations because of mutations or other variation sources, causing generation of differentiations within a population. Since the genome is the source of hereditary and genetic information, these variants induce certain consequences on organisms, leading to genetic diseases, phenotypic abnormalities and more. Therefore, analyzing genomic variants is of crucial importance. General approach to analyse genomic variations include variant calling and variant annotation, where the first one involves the discovery of variants and the latter associates them with genes and functional impacts. Finally, manual curation of the variants is usually performed by utilizing visualization tools. There are several algorithms that use high-throughput sequencing technology to perform each of these tasks. However, one has to deal with each of them separately and the final conclusion is drawn by combining the results. Here we introduce AnGenoV (ANalysis of GENOmic Variants), which combines variant calling, variant annotation and visualization in one single application, thus enabling researchers to spend less time and effort with much higher accuracy. It has a modular structure and serves in a user-friendly graphical interface for easy usage, even for people with minor computational background.
                    <br><br>
                    Using AnGenoV, one can detect single nucleotide polymorphisms (SNPs), small insertions and deletions (Indels) and large structural variations (SVs) using short Illumina reads, as well as third generation long reads (PacBio or Oxford Nanopore). AnGenoV is configured with default state-of-the-art tools, however different algorithms can be added to its pipeline. Associating variations with several possible consequences is another critical point of AnGenoV, which is accessible from the variant annotation module. It incorporates a number of popular variant annotation databases including dbSNP, dbVar and Ensembl Variant Effect Predictor (VEP). On the other hand, we embedded Integrated Genomics Viewer (IGV) into AnGenoV so that selected variants can be visualized by tracking the reads that are mapped to the reference genome. 
                    <br><br>
                    Another important aspect of AnGenoV that increases the accuracy of variant calling is the ability to run several variation discovery tools altogether, merge the results and output the most reliable set of variations. These final variations can be visualized in a user-friendly manner and desired variations can be searched without any UNIX command line knowledge (based on chromosome number, loci, variation type, etc). Then, the variants can be annotated by querying from multiple annotation databases. Finally, the variations can be visualized using IGV by just selecting the variation of interest among the list of possible candidates.
                    <br><br>
                    AnGenoV is implemented using JavaScript and Python and is freely available on https://github.com/SevcanDogramaci/AnGenoV.
                  </p>
                </div>
              </p>

              <hr>

              <p class="text">
                <b>Assessment of the CASP14 Assembly Predictions</b>
                <br>
                <i>Burcu Ozden, Andriy Kryshtafovych and Ezgi Karaca</i>
                <br><br>
                <button class="btn-abstract" data-toggle="collapse" data-target="#collapses2"><i class="fas fa-pencil-alt" aria-hidden="true"></i>Abstract</button>
                <div class="collapse" id="collapses2">
                  <p class="text">
                    In CASP14, 39 research groups submitted more than 2,500 3D models on 22 protein complexes. In general, the community performed well in predicting the fold of the assemblies (for 80% of the targets), though it faced significant challenges in reproducing the native contacts. This is especially the case for the complexes without whole-assembly templates. The leading predictor, BAKER-experimental, used a methodology combining classical techniques (template-based modeling protein docking) with deep learning-based contact predictions and a fold-and-dock approach. The Venclovas team achieved the runner-up position with template-based modeling and docking. By analyzing the target interfaces, we showed that the complexes with depleted charged contacts or dominating hydrophobic interactions were the most challenging ones to predict. We also demonstrated that if AlphaFold2 predictions were at hand, the interface prediction challenge could be alleviated for most of the targets. All in all, it is evident that new approaches are needed for the accurate prediction of assemblies, which undoubtedly will expand on the significant improvements in the tertiary structure prediction field.
                  </p>
                </div>
              </p>

              <hr>

              <p class="text">
                <b>Phylogeny-Aware Amino Acid Substitution Scoring</b>
                <br>
                <i>Nurdan Kuru, Onur Dereli, Emrah Akkoyun, Aylin Bircan, Öznur Taştan and Ogün Adebali</i>
                <br><br>
                <button class="btn-abstract" data-toggle="collapse" data-target="#collapses3"><i class="fas fa-pencil-alt" aria-hidden="true"></i>Abstract</button>
                <div class="collapse" id="collapses3">
                  <p class="text">
                    With the advancement in high throughput sequencing technologies, our ability to detect genetic variation, and to predict the effect of a variant in the clinical diagnosis have been revolutionized. Single nucleotide polymorphisms (SNPs) in coding regions might cause the change of a single amino acid into another in the resulting protein (i.e., missense mutations). These mutations might have no effect on protein function, or it can alter protein function which might result in a disease. Understanding the effect of a missense mutation with respect to whether it has a neutral or disease-causing effect on protein function helps to diagnose rare diseases. Although the cost of genome sequencing has decreased, it is still a challenging task to assess the functional consequences of variations. For this purpose, several conservation-based statistical and machine learning approaches have been proposed in the literature to predict the potential consequence of a variant. SIFT and PolyPhen-2 are the most widely used tools of such approaches. Although these methods do not yield the desired level of accuracy and as a result are not suggested to be used in clinical studies, the clinicians use them to prioritize and reduce the number of variants to be analyzed.
                    <br><br>
                    Here, we introduce a novel phylogeny-dependent probabilistic approach, Phylas (Phylogeny-Aware Amino Acid Substitution Scoring) to predict the functional effects of missense mutations. Our approach exploits the phylogenetic tree information to measure the deleteriousness of a given variant. Independent evolutionary events and phylogenetic relationship among species are driven from the gene-based phylogenetic trees. With the help of ancestral reconstruction, we obtain the probability distribution at each internal node of the phylogenetic tree. Starting from the queried specie, which is human in our algorithm, we travel through the tree and record the probability change for each amino acid. Although the positive change in probability is a result of an alteration, the negative changes are observed as the effect of a substitution that belongs to the previously covered part of the tree. To include each dependent change at once, the negative changes are ignored in the computation. In addition to taking the dependent alterations into account, the effects of independent substitutions observed during the evolutionary time repeatedly should be included. To account for this effect, we present a correction over the score by considering the count of the independent substitutions guaranteeing that the harmfulness of the related alteration is decreased. It has been previously hypothesized that a variant in the human gene is more likely benign when it is observed in closely related species, whereas it is more likely deleterious when it only exists in distant ones. Thus, during the travel through the phylogenetic tree, all positive changes on amino acid probabilities are summed by a weighting approach inversely proportional to the distance between related node of change and human. After completion of the travel, we obtain substitution scores for each of the 20 amino acids for the given position of the query sequence. The normalized resulting values give us the probability of observing any amino acid at the given position of the protein in question. This probability is used to measure the pathogenicity of a possible amino acid substitution. Although the query sequence is human in our experiments, the approach can be easily used for other species by changing the starting point and the direction of the travel through the tree.
                    <br><br>
                    We compare the predictive performance of our algorithm against SIFT and PolyPhen-2. We generated the benchmark datasets by combining variants from Clinvar, Humsavar, Gnomad, and four other datasets proposed in Varibench. Our algorithm outperforms SIFT and PolyPhen-2 in predicting the pathogenicity of missense mutations by improving the AUROC values by 3 and 7% respectively.
                  </p>
                </div>
              </p>

              <hr>

            </div>
          </div>
        </div>

        <div class="row">
          <div class="col-md-3 col-sm-3 col-xs-3 col-4"><p class="time">11:30 - 12:00</p></div>
          <div class="col-md-5 col-sm-5 col-xs-5 col-6"><p class="invited-speaker">Elif Özkırımlı</p></div>
          <div class="col-md-4 col-sm-4 col-xs-4 col-2">
            <button type="button" class="btn-invited" data-toggle="collapse" data-target="#invited3">&nbsp;&#x2b07;&nbsp; Invited Talk</button>
          </div>
        </div>

        <div class="row">
          <div class="col-md-3 col-sm3 col-xs-3 col-4"></div>
          <div class="col-md-9 col-sm9 col-xs-9 col-8"><div id="invited3" class="collapse">
                <p class="text">
                <b>Chemical and Biomedical Language Processing - Challenges and Opportunities</b>
                <br><br>
                <button class="btn-abstract" data-toggle="collapse" data-target="#collapsei3"><i class="fas fa-pencil-alt" aria-hidden="true"></i>Abstract</button>
                <div class="collapse" id="collapsei3">
                  <p class="text">
                    A researcher shares biomedical findings with the scientific community via scientific publications using domain specific language. Human codified representation of biochemicals is also a domain specific language a researcher uses in order to study the mechanism of molecular interactions. Application of natural language processing methodologies for such domain specific languages is often a challenge. However, a more challenging aspect of processing data in these domains is that they do not sample all of the available knowledge space (publications) or molecule space (molecular interactions). This is a pity because most interesting biology occurs at the edge or out-of-distribution. Identifying novel protein - compound pairs or finding rare/new information in publications are both limited by this imbalance problem in data sampling. In this talk, I will summarize our recent work on protein - compound affinity prediction and multilabel text classification of biomedical publications. I will briefly present two novel approaches that aim to address the "needle in a haystack"problem for these two tasks.
                  </p>
                </div>
              </p>
              </div></div>
        </div>

        <div class="row">
          <div class="col-md-3 col-sm-3 col-xs-3 col-4"><p class="time">12:00 - 13:30</p></div>
          <div class="col-md-9 col-sm-9 col-xs-9 col-8">
            
            <button type="button" class="btn-poster" data-toggle="collapse" data-target="#poster1">&nbsp;&nbsp;&nbsp; &#x2b07; Poster Session</button>

            <div id="poster1" class="collapse">
              <br>
              <p class="text">
                <b>(2) DiMA: Protein Sequence Diversity Dynamics Analyser for Viruses</b>
                <br>
                <i>Yongli Hu, Shan Tharanga, Olivo Miotto, Eyyüb Selim Ünlü, Muhammet A. Çelik, Muhammed Miran Öncel, Hilal Hekimoğlu, Muhammad Farhan Sjaugi and Mohammad Asif Khan</i>
              </p>

              <hr>

              <p class="text">
                <b>(3) Convolutional Neural Network Approach to Distinguish and Characterize Tumor Samples Using Gene Expression Data</b>
                <br>
                <i>Büşra Nur Darendeli and Alper Yılmaz</i>
              </p>

              <hr>

              <p class="text">
                <b>(4) CoVrimer for SARS-CoV-2 Primer Prioritization</b>
                <br>
                <i>Merve Vural, Aslınur Aktürk, Mert Demirdizen, Ronaldo Leka, Rana Acar and Özlen Konu</i>
              </p>

              <hr>

              <p class="text">
                <b>(5) RNAseq Based Analysis for Investigation of Crosstalk among Estrogen and Drospirenone Mediated Signaling in Breast Cancer</b>
                <br>
                <i>Merve Vural, Kübra Çalışır, Farid Ahadli, Ronaldo Leka, Burçin Arıcı and Özlen Konu</i>
              </p>

              <hr>

              <p class="text">
                <b>(6) An Agent-Based Model to Evaluate the Effect of Test Kit Usage of Travelers in Sparsely Populated Areas during Pandemics</b>
                <br>
                <i>Baris Balaban, Erdem Berkay Bascura and Ugur Sezerman</i>
              </p>

              <hr>

              <p class="text">
                <b>(7) AnGenoV: A Toolbox for Analysis of Genomic Variants</b>
                <br>
                <i>Sevcan Doğramacı, Mert Doğan, Mehmet Gürsel Arslan, Süleyman Emre Çelik, Tuğba Nur Korkmaz and Arda Söylev</i>
              </p>

              <hr>

              <p class="text">
                <b>(8) On the Path to Reduce Sugar Intake: Sweet Plant Proteins</b>
                <br>
                <i>Nergiz Yuksel, Shokoufeh Yazdanıan Asr and Burcu Kaplan Turkoz</i>
              </p>

              <hr>

              <p class="text">
                <b>(9) Assessment of the CASP14 Assembly Predictions</b>
                <br>
                <i>Burcu Ozden, Ezgi Karaca and Andriy Kryshtafovych</i>
              </p>

              <hr>

              <p class="text">
                <b>(10) AMULET: A Novel Read Count-Based Method for Effective Multiplet Detection from Single Nucleus ATAC-seq Data</b>
                <br>
                <i>Duygu Ucar</i>
              </p>

              <hr>

              <p class="text">
                <b>(11) Candidate Antigen Enrichment Using scRNAseq Data Integration for CAR T Cell Therapy Against Non-Small Cell Lung Cancer</b>
                <br>
                <i>Mert Yıldız and Yasin Kaymaz</i>
              </p>

              <hr>

              <p class="text">
                <b>(12) Novel Approach for Microbiome Meta-Analysis</b>
                <br>
                <i>Farid Musa and Efe Sezgin</i>
              </p>

              <hr>

              <p class="text">
                <b>(13) Robust Prediction of Genetic Mutation Effects by Homology Analysis</b>
                <br>
                <i>Alperen Taciroğlu, Yeşim Aydin Son and Ogün Adebali</i>
              </p>

              <hr>

              <p class="text">
                <b>(14) Support Vector Machine Supported by Disease Ontology (SVM-DO) to Identify mRNA Signatures Discriminating Tumour Cells</b>
                <br>
                <i>Mustafa Erhan Özer, Pemra Özbek Sarıca and Kazım Yalçın Arga</i>
              </p>

              <hr>

              <p class="text">
                <b>(15) Bayesian Networks for Inter-Omics Analysis</b>
                <br>
                <i>Muntadher Jihad and İdil Yet</i>
              </p>

              <hr>

              <p class="text">
                <b>(16) Bioinformatic Analysis of Bifidobacterium Breve TIR Domain</b>
                <br>
                <i>Bahar Bakar, Dicle Dilara Akpinar and Burcu Kaplan Türköz</i>
              </p>

              <hr>

              <p class="text">
                <b>(17) Comparison of the Performances of in Silico Pathogenicity Prediction Tools on Cancer-Related Variants</b>
                <br>
                <i>Metin Yazar and Pemra Özbek Sarıca</i>
              </p>

              <hr>

              <p class="text">
                <b>(18) Phylogeny-Aware Amino Acid Substitution Scoring</b>
                <br>
                <i>Nurdan Kuru, Onur Dereli, Emrah Akkoyun, Aylin Bircan, Öznur Taştan and Ogün Adebali</i>
              </p>

              <hr>

              <p class="text">
                <b>(19) Discovering Coding Lncrnas Using Deep Learning Training Dynamics</b>
                <br>
                <i>Afshan Nabi, Berke Dilekoğlu, Ogün Adebali and Öznur Taştan</i>
              </p>

              <hr>

              <p class="text">
                <b>(20) G-Protein Selective Activation Mechanisms in GPCRs</b>
                <br>
                <i>Berkay Selçuk, İsmail Erol, Serdar Durdağı and Ogün Adebali</i>
              </p>

              <hr>

              <p class="text">
                <b>(21) Analysis of Structural and Functional Impact of SNVs in hAKT1 Gene Using in Silico Tools</b>
                <br>
                <i>Ilayda Uzumcu and Elif Uz Yildirim</i>
              </p>

              <hr>

              <p class="text">
                <b>(22) Bioinformatic Analyses of Heparinase HepIII from Azospirilum brasilanse</b>
                <br>
                <i>Seyhan İçier and Burcu Kaplan Türköz</i>
              </p>

              <hr>

              <p class="text">
                <b>(23) Co-Expression Networks from Transcriptome Data Reveal Molecular Mechanisms Playing Roles in the Progression of Parkinson’s Disease</b>
                <br>
                <i>Tunahan Çakır and Elif Emanetci</i>
              </p>

              <hr>

              <p class="text">
                <b>(24) Identification of Major Depression Related Transcriptional Changes Through Integration of Multiple Datasets</b>
                <br>
                <i>Berkay Selçuk, Tuana Aksu and Ogün Adebali</i>
              </p>

              <hr>

              <p class="text">
                <b>(25) A Story of an Online Internship in Computational Structural Biology</b>
                <br>
                <i>İrem Yılmazbilek and Ezgi Karaca</i>
              </p>

              <hr>

              <p class="text">
                <b>(26) Comparison and Assessment of Speed and Accuracy of AutoDock Vina and AutoDock CrankPep for Short Peptide Docking</b>
                <br>
                <i>Sefer Baday and Numan Nusret Usta</i>
              </p>

              <hr>

              <p class="text">
                <b>(27) miRModuleNet: Detecting miRNA-mRNA Regulatory Modules</b>
                <br>
                <i>Malik Yousef, Gokhan Goy and Burcu Bakir-Gungor</i>
              </p>

              <hr>

              <p class="text">
                <b>(28) Predicting Side Effects of Chemotherapy Using Drug-Induced Gene Expression Profiles and a Random Forest-Based Strategy</b>
                <br>
                <i>Ozlem Ulucan</i>
              </p>

              <hr>

              <p class="text">
                <b>(29) Identification Novel Inhibitors Targeting Putative Dantrolene Binding Site for Ryanodine Receptor 2</b>
                <br>
                <i>Cemil Can Saylan and Sefer Baday</i>
              </p>

              <hr>

              <p class="text">
                <b>(30) Piperidine-Including Natural Drug Discovery for Inhibition Type 4 Pili’s (T4P) in P. aeruginosa and N. meningitidis</b>
                <br>
                <i>Aslıhan Özcan Yöner, Özlem Keskin Özkaya, Berna Sarıyar Akbulut and Pemra Özbek Sarıca</i>
              </p>

              <hr>

              <p class="text">
                <b>(31) Studying Complex Human Diseases Using Time-Series Ancient DNA Data: Obesity and Type 2 Diabetes in Anatolia</b>
                <br>
                <i>İdil Taç, Gülşah Merve Kılınç, Ulaş Işıldak, Kıvılcım Başak Vural, Ezgi Altınışık, Yılmaz Selim Erdal, Mehmet Somel and Füsun Özer</i>
              </p>

              <hr>

              <p class="text">
                <b>(32) Sequence Diversity of Envelope Protein of Dengue Virus Serotype 1</b>
                <br>
                <i>Gökçen Şahin, Li Chuin Chong, Erdem Aybek and Mohammad Asif Khan</i>
              </p>

              <hr>

              <p class="text">
                <b>(33) Investigating Potential Interplay Between R-Loops and Nucleotide Excision Repair</b>
                <br>
                <i>Sezgi Kaya and Ogün Adebali</i>
              </p>

              <hr>

              <p class="text">
                <b>(34) PersonaDrive: A Computational Approach for Prioritization of Patient-specific Cancer Drivers</b>
                <br>
                <i>Cesim Erten, Aissa Houdjedj, Hilal Kazan and Ahmed Amine Taleb Bahmed</i>
              </p>

              <hr>

              <p class="text">
                <b>(35) PROT-ON: A Python Package for Redesigning the Protein-Protein Interfaces by Using EvoEF1</b>
                <br>
                <i>Mehdi Koşaca, Ayşe Berçin Barlas and Ezgi Karaca</i>
              </p>

              <hr>

              <p class="text">
                <b>(36) GeNetKEGG: Gene Expression based KEGG PathWay Grouping and Ranking</b>
                <br>
                <i>Malik Yousef, Fatma Ozdemir, Amhar Jabeer, Jens Allmer and Burcu Bakir-Gungor</i>
              </p>

              <hr>

              <p class="text">
                <b>(37) How Epstein-Barr Virus Envelope Glycoprotein gp350 Tricks the CR2? A Molecular Dynamics Study.</b>
                <br>
                <i>Ilgaz Taştekil, Cansu Yay, Nursena Keskin, Elif Naz Bingöl and Pemra Ozbek Sarica</i>
              </p>

              <hr>

              <p class="text">
                <b>(38) Metabolic Network-Driven Analysis of Yeast Metabolic Cycle through the Incorporation of RNA-seq and ATAC-seq Datasets</b>
                <br>
                <i>Müberra Fatma Cesur, Tunahan Çakır and Pınar Pir</i>
              </p>

              <hr>

              <p class="text">
                <b>(39) Peptide - Gold (111) Interactions: Mechanisms and Design</b>
                <br>
                <i>Didem Özkaya, Büşra Demir, Çağlanaz Akın, Zeynep Köker and Ersin Emre Ören</i>
              </p>

              <hr>

              <p class="text">
                <b>(40) The Mutation Profile of SARS-CoV-2 Is Primarily Shaped by the Host Antiviral Defense</b>
                <br>
                <i>Cem Azgari, Zeynep Kılınç, Berk Turhan, Defne Çirci and Ogün Adebali</i>
              </p>

              <hr>

              <p class="text">
                <b>(41) Application of Machine Learning Algorithm for The Accurate Diagnosis of Breast Cancer</b>
                <br>
                <i>Rumeysa Fayetörbay and Uğur Sezerman</i>
              </p>

              <hr>

              <p class="text">
                <b>(42) A Time-Efficient and User-Friendly Tool for Molecular Dynamics Analysis</b>
                <br>
                <i>Halil İbrahim Özdemir, Elif Naz Bingöl and Pemra Özbek-Sarıca</i>
              </p>

              <hr>

            </div>
          </div>
        </div>

        <div class="row">
          <div class="col-md-3 col-sm-3 col-xs-3 col-4"><p class="time">13:30 - 14:30</p></div>
          <div class="col-md-9 col-sm-9 col-xs-9 col-8">
            <button type="button" class="btn" data-toggle="collapse" data-target="#talk2">&nbsp;&#x2b07;&nbsp; Selected Talk Session - 2</button>
              <div id="talk2" class="collapse">

                <p class="text">

                <br>
                <b>Discovering Coding Lncrnas Using Deep Learning Training Dynamics</b>
                <br>
                <i>Afshan Nabi, Berke Dilekoğlu, Ogün Adebali and Öznur Taştan</i>
                <br><br>
                <button class="btn-abstract" data-toggle="collapse" data-target="#collapses4"><i class="fas fa-pencil-alt" aria-hidden="true"></i>Abstract</button>
                <div class="collapse" id="collapses4">
                  <p class="text">
                    Genome-wide transcriptome analyses have revealed that the vast majority of the human genome is transcribed; but only 2% of the human genome is annotated as protein coding. A considerable fraction of transcripts are annotated as ncRNAs and lncRNAS constitute the largest category of ncRNAs. While lncRNAs studied are known to play vital roles in cellular processes, the functions of most lncRNAs remain unknown. Moreover, although lncRNAs - by definition- do not code for proteins, recent studies have shown that short the open reading frames (sORFs) within some lncRNAs are translated into micropeptides of a median length of 23 amino acids. The translation events of lncRNAs were overlooked previously because the open reading frames (ORFs) present in lncRNAs do not meet the conventional criteria of an ORF: that it encodes at least 100 amino acids in eukaryotes. Despite this, recent studies have shown that micropeptides translated from lncRNAs perform vital functions across species, including bacteria, flies and humans. Therefore, identifying misannotated lncRNAs is a necessary step towards the functional 1 characterization of this large class of transcripts.
                    <br><br>
                    We present a framework that leverages deep learning models’ training dynamics to determine whether a given lncRNA transcript in the dataset is misannotated. In the first step, we train convolutional neural network (CNN), long short term memory (LSTM) and Transformer architectures to predict whether a given nucleotide sequence is non-coding or coding. Each input RNA sequence is represented with 3-mer ‘words’ are obtained by using a window that slides by 1 nucleotide at each step and for each 3-mer ‘word’, 100-dimensional embeddings are used as input. Our models learn to distinguish between coding and non-coding RNAs with average AUC scores >91%. In the second step, we inspect the training dynamics of these deep sequence classifiers to identify possible misannotated lncRNAs. By inspecting lncRNAs where the model consistently and with high confidence predicts as coding through all training epochs, we identify the possibly misannotated candidates. Through this inspection, we identify candidate lncRNAs. Our results show a significant overlap with previous methods that use riboseq data to identify misannotated lncRNAs as well as with a set of experimentally validated misannotated lncRNAs. Moreover, we search proteins similar in sequence to the candidates and curate a subset with high similarity to known proteins.
                    <br><br>
                    This work represents the first instance where deep learning model training dynamics are successfully applied to identify misannotated lncRNAs from nucleotide sequences. This approach can be applied to better curate datasets for training coding potential prediction models and can be applied alongside ribo-seq data to identify misannotated lncRNAs with high confidence. </p>
                </div>
              </p>

              <hr>

              <p class="text">
                <b>Piperidine-including natural drug discovery for inhibition Type 4 Pili’s (T4P) in P. aeruginosa and N. meningitidis</b>
                <br>
                <i>Aslıhan Özcan Yöner, Özlem Keskin Özkaya, Berna Sarıyar Akbulut and Pemra Özbek Sarıca</i>
                <br><br>
                <button class="btn-abstract" data-toggle="collapse" data-target="#collapses5"><i class="fas fa-pencil-alt" aria-hidden="true"></i>Abstract</button>
                <div class="collapse" id="collapses5">
                  <p class="text">
                    The rapidly increasing resistance to available antibiotics and the reduced rate in new antibiotic discovery lead to different therapeutic approaches. An attractive strategy is to identify molecules that target bacterial virulence as an alternative to traditional antibiotics with low efficacies. Anti-virulence therapies cleanse the pathogens from their weapons instead of killing the pathogens that cause infections in humans [1], [2].
                    <br><br>
                    The current work undertakes the effort to target the type 4 pili (T4P) in P. aeruginosa and N. meningitidis. T4P is an important virulence factor in both and is the main target of this study. The selected two microorganisms are among the World Health Organization’s list of pathogenic bacteria for which there is urgence to develop new therapeutics. To this end, PilB of P. aeruginosa and PilF of N. meningitidis have been studied in detail. As a novel strategy, piperidine-containing natural product solutions were screened for their inhibition of pilB and pilE. Piperidine is an incredibly significant building block in the production of pharmacological substances and is the most regularly utilized heterocycle among US FDA approved medications [3].
                    <br><br>
                    The strategy followed in this work started with homology modelling of pilB and pilE of P. aeruginosa and N. meningitidis, respectively, since their structures were not available. The binding sites in the target structures were determined by metaPocket 2.0. Here the ADP binding regions have been observed to play an important role in the to inhibit T4P. As the drug database, COlleCtion of Open Natural prodUcTs (COCONUT) resource was used and for filtration FAF-Drugs4 was used. Finally, in order to select natural products to inhibit T4P by binding pilB and pilE, virtual library screening was performed. Ligands with binding energies better than -9.0kcal/mol for both structures were accepted to be potential inhibitors. Molecules selected in this work might have a potential to be used in novel therapeutic applications in combination with existing drugs and/or other virulence factor inhibitors. 
                  </p>
                </div>
              </p>

              <hr>

              <p class="text">
                <b>PersonaDrive: A Computational Approach for Prioritization of Patient-specific Cancer Drivers</b>
                <br>
                <i>Cesim Erten, Aissa Houdjedj, Hilal Kazan and Ahmed Amine Taleb Bahmed</i>
                <br><br>
                <button class="btn-abstract" data-toggle="collapse" data-target="#collapses6"><i class="fas fa-pencil-alt" aria-hidden="true"></i>Abstract</button>
                <div class="collapse" id="collapses6">
                  <p class="text">
                    A major challenge in cancer genomics is to distinguish the driver mutations that are causally linked to cancer from passenger mutations that do not contribute to cancer development. The majority of the methods proposed for this problem provide a single driver gene list for the entire cohort of patients. On the other hand, it is well-known that the mutation profiles of patients from the same cancer type show a high degree of heterogeneity. Since each patient has a distinct set of driver genes, a more ideal approach is to identify patient-specific drivers.
                    <br><br>
                    In this study, we propose a novel method that integrates genomic data, biological pathways, and protein connectivity information for personalized identification of driver genes. The method is formulated on a personalized bipartite graph which consists of the mutated genes of the patient in one partition, and the set of patient-specific dysregulated genes in the other. Our approach provides a personalized ranking of the mutated genes of a patient based on the sum of weighted ‘pairwise pathway coverage’ scores across all the patients, where an appropriate pairwise patient similarity score with respect to the sets of dysregulated genes of the pair is employed as a weighting factor of pathway coverage scores of mutant-dysregulated pairs. We compare our method against three state-of-the-art patient-specific cancer gene prioritization methods; Prodigy [1], SCS [2] and DawnRank [3]. The comparisons are with respect to a novel evaluation method which takes into account the personalized nature of the problem; different from previous evaluation methods employed in literature, we assume the results are reported as average values for the entire cohort as a function of the top k ranked genes, where k is dependent on the size of the personalized reference set and individuals with less than k ranked genes are excluded from the evaluation. Two main data sets are considered for the evaluations; TCGA and cell-line data (DepMAP [5]) for colon and lung cancers. For the TCGA data, the reference sets are defined based on the overlaps of mutated genes of the patient and the Cancer Gene Census (CGC) [6], Network of Cancer Genes (NCG) [7], and CancerMine[8] databases of known driver genes. For the cell-line data, we define novel reference gene sets by compiling the targets of drugs that are found to be sensitive based on data from GDSC [4] and DepMAP [5] databases. We show that our approach outperforms the existing alternatives for both the TCGA and cell-line evaluations. Additionally, we show that the KEGG/Reactome pathways enriched in our ranked genes and those that are enriched in cell lines reference sets overlap significantly when compared to the overlaps achieved by the rankings of the alternative methods. Fig 1 shows the performance of all methods on TCGA and CCLE cohorts. The findings of our approach can lead to the development of personalized treatments and therapies.
                    <br><br>
                    References
                    <br>
                    [1] Gal Dinstag, et al. (2020) Bioinformatics. 1831–1839
                    <br>
                    [2] Guo WF, et al. (2018). Bioinformatics. 1;34(11):1893-1903.
                    <br>
                    [3] Hou, J.P., Ma, J. (2014). Genome Med 6, 56.
                    <br>
                    [4] Wanjuan Yang, et al. (2013). Nucleic Acids Research, V.41, Pages D955–D961.
                    <br>
                    [5] Steven M Corsello, et al. (2019). bioRxiv.
                    <br>
                    [6] John G Tate, et al. (2019). Nucleic Acids Research, V.47, P.D941–D947.
                    <br>
                    [7] Repana D., et al. (2019). Genome Biol.
                    <br>
                    [8] Lever J. et al. (2019). Nat Methods, 16:505–507
                  </p>
                </div>
              </p>

              <hr>
            </div>
          </div>
        </div>

        <div class="row">
          <div class="col-md-3 col-sm-3 col-xs-3 col-4"><p class="time">14:30 - 15:00</p></div>
          <div class="col-md-5 col-sm-5 col-xs-5 col-6"><p class="invited-speaker">Irene Papatheodorou</p></div>
          <div class="col-md-4 col-sm-4 col-xs-4 col-2">
            <button type="button" class="btn-invited" data-toggle="collapse" data-target="#invited4">&nbsp;&#x2b07;&nbsp; Invited Talk</button>
          </div>
        </div>

        <div class="row">
          <div class="col-md-3 col-sm3 col-xs-3 col-4"></div>
          <div class="col-md-9 col-sm9 col-xs-9 col-8"><div id="invited4" class="collapse">
                <p class="text">
                <b>Expression Atlas: Gene expression across cells, tissues, species</b>
                <br><br>
                <button class="btn-abstract" data-toggle="collapse" data-target="#collapsei4"><i class="fas fa-pencil-alt" aria-hidden="true"></i>Abstract</button>
                <div class="collapse" id="collapsei4">
                  <p class="text">
                    The Gene Expression Group at EMBL-EBI develops and multiple functional genomics resources from data submission to analysis and visualisation of bulk and single cell sequencing data.
                    <br><br>
                    In this talk we will describe the resource Expression Atlas and its single cell component Single Cell Expression Atlas (www.ebi.ac.uk/gxa), our resource for searching gene expression across cell types, tissues and species from re-analyses of publicly available bulk and single cell RNA-Seq (scRNA-Seq) studies, including those from landmark projects. As of September 2021, Expression Atlas contains over 4000 datasets, across 65 species and over 5 million cells. Users can search for genes across these datasets and filter the results for particular cell types or tissues, identify in what conditions and populations a gene can act as a marker gene.
                    <br><br>
                    In addition, we will describe our flexible and scalable, standardised analysis pipelines. These are a set of flexible, scalable and interoperable analysis workflows for the analysis of scRNA-Seq data sets, providing access to data sets from the Single Cell Expression Atlas.
                    <br><br>
                    Finally, we will describe our current work to enable automated cell type annotation, power more informative searches, as well as intuitive visualisations using zoomable organ-specific anatomograms.
                  </p>
                </div>
              </p>
              </div></div>
        </div>

        <div class="row">
          <div class="col-md-3 col-sm-3 col-xs-3 col-4"><p class="time">15:00 - 15:20</p></div>
          <div class="col-md-9 col-sm-9 col-xs-9 col-8"><p class="activity">Break</p></div>
        </div>

        <div class="row">
          <div class="col-md-3 col-sm-3 col-xs-3 col-4"><p class="time">15:20 - 16:20</p></div>
          <div class="col-md-9 col-sm-9 col-xs-9 col-8">
            <button type="button" class="btn" data-toggle="collapse" data-target="#talk3">&nbsp;&#x2b07;&nbsp; Selected Talk Session - 3</button>
              <div id="talk3" class="collapse">
                <p class="text">

                <br>
                <b>Metabolic network-driven analysis of yeast metabolic cycle through the incorporation of RNA-seq and ATAC-seq datasets</b>
                <br>
                <i>Müberra Fatma Cesur, Tunahan Çakır and Pınar Pir</i>
                <br><br>
                <button class="btn-abstract" data-toggle="collapse" data-target="#collapses7"><i class="fas fa-pencil-alt" aria-hidden="true"></i>Abstract</button>
                <div class="collapse" id="collapses7">
                  <p class="text">
                    Saccharomyces cerevisiae, which is a well-established model organism in many industrial and medical applications, undergoes robust oscillations to regulate its physiology for adaptation and survival under nutrient-limited conditions. The rhythmic alterations in gene expression pattern and cell metabolism coordinate responding to environmental cues. Yeast metabolic cycle (YMC) is a remarkable example of the coordinated and dynamic yeast behaviour, which is regulated through metabolic oscillations. It is divided into three phases based on periodic alterations in gene expression across varying oxygen consumption levels: quiescence-related reductive charging (RC) phase, growth-related oxidative (OX) phase, and cell division-related reductive building (RB) phase. Thus, YMC tracks the life cycle of yeast via an interplay among growth, proliferation, and quiescent phases.
                    <br><br>
                    Genome-scale metabolic network (GMN) models have been extensively used platforms to analyse yeast metabolism since 2003. They are stoichiometry-based mathematical representations of metabolism with all known chemical reactions, metabolites, and genes. GMN models provide a powerful platform for the systems-based understanding of metabolic processes within an organism. To date, different omics data integrated models have been developed for phenotypic characterizations and metabolic engineering. Despite the common use of transcriptome in the contextualization of GMN models, incorporation of epigenetic information is still a gap in the field of metabolic modelling. Besides, a clear interaction between metabolism and epigenetics was highlighted in many studies. Here, we investigated the contribution of combinatory use of transcriptomic and epigenomic information in the simulation of cellular metabolism via a recent yeast model, Yeast8 (3,991 reactions, 2,691 metabolites, and 1,147 genes). To this aim, we first employed hierarchical clustering for both RNA-seq and ATAC-seq datasets dedicated to each YMC phase. Thus, the pathways associated with each YMC phase were identified (data-based approach). We subsequently reconstructed diverse GMN models through mapping these datasets in both individual and combinatorial fashions. This facilitated the simulation of early RC, mid OX, and late RB phases. Thus, we evaluated the performance of each model using the experimental flux data derived from 13C metabolic flux analysis. We also characterized differential flux profiles and pathways through comparative analyses (model-based approach). Lastly, we compared the results obtained via data- and model-based approaches to each other and validated based on the literature.
                    <br><br>
                    Comparative analysis of the predicted and measured fluxes revealed that the use of ATAC-seq data considerably improved model performances. The pathways dedicated to each YMC phase were elucidated using data- and model-based approaches. As expected, over-representation of the growth-related processes (e.g., biosynthesis of amino acids and nucleotides) and tricarboxylic acid cycle were shown in mid OX phase. On the other hand, early RC and late RB phases were found to exhibit similar characteristics. Over-representation of various glycolytic processes, NADP metabolism, and pentose-phosphate shunt were determined in late RB phase in agreement with literature. To our knowledge, this is the first attempt to use chromatin accessibility data in the reconstruction of context-specific GMN models, despite the increasing popularity of ATAC-seq method. Thus, we demonstrated that integration of epigenomic data with transcriptomic profiling can pave the way for more realistic metabolic simulations.
                  </p>
                </div>
              </p>

              <hr>

              <p class="text">
                <b>mirDisNet: A novel approach for cancer classification using mir-disease associations</b>
                <br>
                <i>Amhar Jabeer, Burcu Bakir-Gungor and Malik Yousef</i>
                <br><br>
                <button class="btn-abstract" data-toggle="collapse" data-target="#collapses8"><i class="fas fa-pencil-alt" aria-hidden="true"></i>Abstract</button>
                <div class="collapse" id="collapses8">
                  <p class="text">
                    miRNAs (microRNAs) are a family of short non-coding RNAs that regulate gene expression post-transcriptionally in diverse species. They repress protein production by translational silencing, binding to the 3'-UTR (untranslated region) of their target mRNAs, and destabilizing them. Growing evidence shows that miRNAs exhibit a variety of crucial regulatory functions in all mammals; and because of their potential role related to cell growth, development, and differentiation, while being associated with a wide variety of human diseases. They have been proposed to be good candidates for cancer therapy since they have been associated with cancer biology: metastasis, angiogenesis, and proliferation. Conversely, considering the inherent time-consuming and expensive method of traditional in vitro experiments, the need for feasible and efficient computational methods to predict miRNA and diseases association have become apparent. We propose mirDisNet, a novel approach that detects the biomarker of miRNAs genes that is associated with diseases. In mirDisNet, biological domain data that incorporates the knowledge about miRNAs association with disease is used to serve as the grouping function for the tool. Each of the groups created have a disease name with a corresponding set of miRNAs related to the disease. We ranked the groups by scoring them on their importance in the two-class classification task. By integrating miRNA-disease associations, mirDisNet showed promising results of 95% in accuracy, 92% in sensitivity, 96% in specificity, and 98% in AUC across 11 datasets obtained from TCGA. Additionally, the most significant miRNAs and disease groups which were ranked by robustrankaggreg, were validated by external datasets, databases, and literature. We hypothesize that mirDisNet has the prospect to understand disease prognosis as well as diagnosis by finding potential biomarkers and disease relationship networks.
                  </p>
                </div>
              </p>

              <hr>

              <p class="text">
                <b>Functional Stratification of Small Molecule Drugs through Integrated Network Similarity</b>
                <br>
                <i>Seyma Unsal Beyge and Nurcan Tuncbag</i>
                <br><br>
                <button class="btn-abstract" data-toggle="collapse" data-target="#collapses9"><i class="fas fa-pencil-alt" aria-hidden="true"></i>Abstract</button>
                <div class="collapse" id="collapses9">
                  <p class="text">
                    Recovery of an optimal cancer treatment strategy is challenging due to the inter- and intra-heterogeneity of tumor samples. Modulations in signaling pathways and interactions between various bio-entities are critically important in the multi-stage tumor cell formation. Hence, multi-omic data integration is vital for understanding the molecular interactions happening in the cancer cell and for development of an optimum treatment strategy. Since development and approval of new drugs is both expensive and time-consuming, drug repurposing is an advantageous strategy in cancer treatment. Classification of currently available therapeutic agents is important but also a complex procedure and necessary to index possible drugs for drug repurposing approaches. Moreover, determination of molecular mechanisms of available drugs in different cancer types deciphers the possibilities in drug repurposing given the heterogeneity of cancer. Conventionally, drugs are classified based on their primary targets, therapeutic actions, target specificity, nature of interaction, molecular type and chemical structure similarity. The effects of small drug molecules are highly dependent on the cellular and physiological factors. Many drugs have multiple targets with variable binding affinities. Depending on the features selected, groups of similar drugs may change. Even though two drugs are present in the same group, they may modulate different signaling mechanisms within the cell.
                    <br><br>
                    In this study, the transcriptomic and phosphoproteomic data of cell lines treated with small molecule drugs are used to reconstruct networks by integrating the data with drug targetome and human interactome. Data integration is a challenging task since the proper integration method changes depending on the nature of the data and the network reconstruction principles. In this study transcriptomic data is used to back trail the regulatory elements that acts on the experimental hits. Phosphoproteomic hits allowed the selection of functionally active proteins that may be closely related to the drug modulatory effects. Also, human interactome is referenced by the network reconstruction software, Omics Integrator, to map the seed proteins and find the optimum connected subnetwork. Human interactome is known with its incompleteness and also it has bias to well-studied proteins. Therefore, it has many false negatives and false positives. In this study, human interactome is processed to eliminate these drawbacks. First, high-degree nodes eliminated and for each cell line – drug condition, it is filtered for very low expressed genes by the aid of associated transcriptomic data. After, each interactome is enriched with a link prediction approach followed by localization filters. The final processed interactome used for each cell line – drug condition has turned into both tissue and drug specific.
                    <br><br>
                    Total of 250 cell line and drug specific networks are reconstructed covering 70 drugs and six cell lines. A rigorous topological and pathway analysis of these reconstructed networks provided insight into the drug modulations occurring in different cell lines, including mostly cancer models. It is found that chemically and functionally different drugs may modulate overlapping networks. Moreover, the target selectivity of the drug is an important factor leading to separate networks for drugs with same mode of action. Network-based analysis coupled with multi-omic data integration helped to reveal cell line and drug specific hidden modulated pathways such that drugs having overlapping networks generally modulate transcriptional misregulation in cancer pathway. Next, topological distance and active pathways of drug networks may guide the use of efficient drug combinations. Finally, separation between networks of a drug across cell lines can help to infer their resistance or sensitivity or no response to that drug.
                  </p>
                </div>
              </p>

              <hr>
            </div>
          </div>
        </div>
        
        <div class="row">
          <div class="col-md-3 col-sm-3 col-xs-3 col-4"><p class="time">16:20 - 17:20</p></div>
          <div class="col-md-5 col-sm-5 col-xs-5 col-6"><p class="keynote-speaker">Ziv Bar-Joseph</p></div>
          <div class="col-md-4 col-sm-4 col-xs-4 col-2">
            <button type="button" class="btn-keynote" data-toggle="collapse" data-target="#keynote1">&nbsp;&#x2b07;&nbsp; Keynote Talk</button>
          </div>
        </div>
        
        <div class="row">
          <div class="col-md-3 col-sm3 col-xs-3 col-4"></div>
          <div class="col-md-9 col-sm9 col-xs-9 col-8"><div id="keynote1" class="collapse">
                
                <p class="text">
                <b>Reconstructing dynamic regulatory and signaling networks from time-series single-cell data</b>
                <br><br>
                <button class="btn-abstract" data-toggle="collapse" data-target="#collapsek1"><i class="fas fa-pencil-alt" aria-hidden="true"></i>Abstract</button>
                <div class="collapse" id="collapsek1">
                  <p class="text"> Biological processes, including those involved in immune response, disease progression and development, are often dynamic. To fully understand and model regulatory and signaling networks that are activated as part of these processes requires the integration of static and times series bulk and single cell data. I will discuss statistical and active learning methods for designing experiments for studying such systems and methods using continuous state hidden Markov models for the analysis and integration of profiled data. An application of these methods for  improving protocols for the differentiation of iPSCs to lung cells and for predicting cell-type interactions in liver organoid development will be discussed. </p>
                </div>
              </p>

            </div>
          </div>
        </div>
      </div> 

      <div class="col-md-6 col-sm-12 col-xs-12 col-12">
        <h1>Day 2</h1>
        
        <hr>

        <div class="row">
          <div class="col-md-3 col-sm-3 col-xs-3 col-4"><p class="time">09:00 - 09:30</p></div>
          <div class="col-md-5 col-sm-5 col-xs-5 col-6"><p class="invited-speaker">Niranjan Nagarajan</p></div>
          <div class="col-md-4 col-sm-4 col-xs-4 col-2">
            <button type="button" class="btn-invited" data-toggle="collapse" data-target="#invited5">&nbsp;&#x2b07;&nbsp; Invited Talk</button>
          </div>
        </div>

        <div class="row">
          <div class="col-md-3 col-sm3 col-xs-3 col-4"></div>
          <div class="col-md-9 col-sm9 col-xs-9 col-8"><div id="invited5" class="collapse">
                <p class="text">
                <b>Assembling and modelling complex microbiomes mediating antibiotic-resistance transfer</b>
                <br><br>
                <button class="btn-abstract" data-toggle="collapse" data-target="#collapsei5"><i class="fas fa-pencil-alt" aria-hidden="true"></i>Abstract</button>
                <div class="collapse" id="collapsei5">
                  <p class="text">
                    Human and environmental microbial communities mediating host-pathogen interactions often have complex genetic architectures and dynamics. Unravelling these needs new approaches for metagenome assembly at the strain level and microbiome modelling from limited relative abundance profiles. We propose a hybrid assembly framework that leverages long read sequencing to generate high-quality, near-complete strain genomes from complex metagenomes (OPERA-MS [1]). Applying this approach to human and environmental communities enabled recovery of 100s of novel genomes, plasmid and phage sequences, direct analysis of transmission patterns and investigation of antibiotic resistance gene combinations [1, 2]. Furthermore, we show how microbial community dynamics can be modelled accurately from sparse relative abundance data (BEEM [3]), providing insights into pathogen-commensal interactions in skin dermotypes [4]. Data from several studies tracking the emergence and transmission of multi-drug resistant pathogens across environmental and human microbiomes will be used to illustrate the utility of these methods [2,5].
                    <br><br>
                    References
                    <br>
                    [1] Bertrand et al. “Hybrid metagenomic assembly enables high-resolution analysis of resistance determinants and mobile elements in human microbiomes. Nature Biotechnology 2019
                    <br>
                    [2] Chng et al. ”Cartography of opportunistic pathogens and antibiotic resistance genes in a tertiary hospital environment." Nature Medicine 2020
                    <br>
                    [3] Li et al. “An expectation-maximization algorithm enables accurate ecological modeling using longitudinal microbiome sequencing data." Microbiome 2019
                    <br>
                    [4] Tay et al. “Atopic dermatitis microbiomes stratify intoecologic dermotypes enabling microbialvirulence and disease severity.” Journal of Allergy and Clinical Immunology 2020
                    <br>
                    [5] Cheng et al, “Metagenome-wide association analysis identifies microbial determinants of post-antibiotic ecological recovery in the gut.” Nature Ecology & Evolution 2020
                  </p>
                </div>
              </p>
              </div></div>
        </div>

        <div class="row">
          <div class="col-md-3 col-sm-3 col-xs-3 col-4"><p class="time">09:30 - 10:30</p></div>
          <div class="col-md-9 col-sm-9 col-xs-9 col-8">
            <button type="button" class="btn" data-toggle="collapse" data-target="#talk4">&nbsp;&#x2b07;&nbsp; Selected Talk Session - 4</button>
            <div id="talk4" class="collapse">

                <p class="text">

                <br>
                <b>Towards integrative mechanistic models of mammalian cell responses to extracellular perturbations: growth factors, hormones, and cytokines</b>
                <br>
                <i>Cemal Erdem, Sean M. Gross, Laura M. Heiser and Marc R. Birtwistle</i>
                <br><br>
                <button class="btn-abstract" data-toggle="collapse" data-target="#collapses10"><i class="fas fa-pencil-alt" aria-hidden="true"></i>Abstract</button>
                <div class="collapse" id="collapses10">
                  <p class="text">
                    A critical missing capability in current cancer research is the ability to predict how a particular single cancer cell will respond to microenvironmental cues or a drug cocktail. Yet, it is not even possible to perform this task well for normal healthy cells. This work builds on the hypothesis that first principles, mechanistic models of how cells respond to different perturbagens, will ultimately improve drug combination response predictions. However, building such single-cell models of complex, large-scale, and incompletely understood systems remains an extremely challenging task. To address this issue, we defined an open-source pipeline for scalable, single-cell mechanistic modeling from simple, annotated input files (structured lists of species, parameters, and reaction types). The input files are converted into an SBML (Systems Biology Markup Language) model file. Using this pipeline, we:
                    <br>
                    1.Re-created one of the largest pan-cancer signaling models in the literature (774 species, 141 genes, 8 ligands, 2400 reactions)
                    <br>
                    2.Enlarged the model to include Interferon-γ (IFNγ) signaling pathway (950 species, 150 genes, 9 ligands, 2500 reactions)
                    <br>
                    3.Re-parametrized the model to test and prioritize candidate mechanisms for experimental observations
                    <br><br>
                    Specifically, we used the enlarged model to test alternative mechanistic hypotheses for the experimental observations that IFNγ inhibits epidermal growth factor (EGF)-induced cell proliferation. We ran stochastic single-cell simulations for two different crosstalk mechanisms and looked at the number of cycling cells in each case. Our model-based analysis suggested, and experiments support that these observations are better explained by IFNγ-induced SOCS1 expression sequestering activated EGF receptors, thereby downregulating AKT activity, as opposed to direct IFNγ-induced upregulation of p21 expression. Finally, our new modeling format is available online (github.com/birtwistlelab/SPARCED) and compatible with high-performance (Kubernetes) computing platforms, enabling us to study virtual cell population responses. Overall, our new model enables easy modification of large mechanistic models and simulation of thousands of single-cell responses to multiple ligands and drug combinations.
                  </p>
                </div>
              </p>

              <hr>

              <p class="text">
                <b>Runs of homozygosity show that human inbreeding has decreased in time through the Holocene</b>
                <br>
                <i>Kanat Gürün, Francisco C. Ceballos, Ezgi Altınışık, Hasan Can Gemici, Cansu Karamurat, Dilek Koptekin, Kıvılcım Başak Vural, Igor Mapelli, Ekin Sağlıcan, Elif Sürer, Yılmaz Selim Erdal, Anders Götherström, Füsun Özer, Çiğdem Atakuman and Mehmet Somel</i>
                <br><br>
                <button class="btn-abstract" data-toggle="collapse" data-target="#collapses11"><i class="fas fa-pencil-alt" aria-hidden="true"></i>Abstract</button>
                <div class="collapse" id="collapses11">
                  <p class="text">
                    Runs of homozygosity (ROH) are long homozygous stretches of the genome, presence of which indicates inbreeding due to small population sizes and genetic drift, and/or mating between close relatives, i.e., consanguinity [1]. We developed a method to optimize the parameters of PLINK to detect ROH, which relies on a model-free, observational approach that does not require a reference panel [2]. We were able to tune ROH calling parameters to suit low genomic coverages and correct for ROH overestimation. Our method works efficiently down to 3x SNP coverage and reliably calls ROH > 1 Mb in genomes across >1 million SNPs [3]. We confirmed the power and accuracy using simulations and by comparison with a recently published method, which relies on a reference haplotype panel [4]. We used our approach to study the controversial history of human inbreeding by systematically analyzing for the first time the ROH levels in 411 published ancient genomes of the last 15,000 years from West and Central Eurasia. The Neolithic Transition to food production and the development of sedentary and/or agricultural societies may have influenced overall inbreeding levels, relative to those of hunter-gatherer communities. This transition could have had opposite effects on average ROH levels: ROH might decrease by increasing population size [5], or ROH might increase due to higher consanguinity and endogamy in farming communities compared to forager communities [6-7]. We estimated the genomic inbreeding coefficient FROH per genome as the sum of ROH >1.5 [8] and showed that the frequency of inbreeding, as measured by FROH, has decreased over time throughout the Holocene. This result was robust to the SNP list used and was reproducible in downsampling experiments. The result was also supported by a multiple regression model that includes time (age), cultural groups, and technical covariates. Some ancient individuals showed high FROH, but were rare in the sample and they included both hunter-gatherers and farmers. The main shift in FROH happens after the Neolithic, but the trend has since continued, indicating a population size effect on ROH and inbreeding prevalence. Post-Neolithic increase in population admixture [9] may also play role. We further show that most inbreeding in our historical sample can be attributed to small population size and drift, instead of consanguinity. Such high drift individuals were mainly hunter-gatherers. Extreme consanguineous matings did occur, but were rare and only observed among agriculturalist members of farming societies in our sample, in line with ethnographic work [6-7]. Despite the lack of evidence for common consanguinity in our ancient sample, consanguineous traditions are today prevalent in various modern-day Eurasian societies, suggesting that such practices may have become widespread within the last few millennia.
                    <br><br>
                    References
                    <br>
                    [1] Ceballos et al. 2018 Nat Rev Genet;
                    <br>
                    [2] Chang et al. 2015 Gigascience;
                    <br>
                    [3] Ceballos et al. 2020 Biorxiv;
                    <br>
                    [4] Ringbauer et al. 2020 Biorxiv;
                    <br>
                    [5] Gignoux et al. 2011 PNAS;
                    <br>
                    [6] Walker 2014 Evol & Human Behav;
                    <br>
                    [7] Hill et al 2011 Science;
                    <br>
                    [8] McQuillan et al. 2008 AJHG;
                    <br>
                    [9] Lazaridis et al. 2016 Nature
                  </p>
                </div>
              </p>

              <hr>

              <p class="text">
                <b>ProFAB – Open Protein Functional Annotation Benchmark</b>
                <br>
                <i>Ahmet Samet Özdilek, Ahmet Atakan, Tunca Doğan, Rengül Çetin-Atalay, Mehmet Volkan Atalay and Ahmet Süreyya Rifaioğlu</i>
                <br><br>
                <button class="btn-abstract" data-toggle="collapse" data-target="#collapses12"><i class="fas fa-pencil-alt" aria-hidden="true"></i>Abstract</button>
                <div class="collapse" id="collapses12">
                  <p class="text">
                  As the number of protein sequences in protein databases increases, accurate computational methods are required to annotate the available data. For this purpose, several machine learning methods have been proposed in recent years [1]. However, two main issues in the evaluation of computational prediction methods are the construction of reliable positive and their negative training/validation datasets and the fair evaluation of performances based on predefined experimental settings. Recently, several benchmarking platforms have been proposed in various fields to overcome similar issues. For example, Therapeutics Data Commons provides ready-to-use biomedical datasets for drug, toxicity, screening, antibody development along with the appropriate evaluation metrics [2]. Open Graph Benchmark is a platform that provides social and biological network datasets, and experimental settings for fair comparison of algorithms [3]. In the field of protein function prediction, Critical Assessment of Functional Annotation (CAFA) challenge [4] is an important initiative where the aim is to evaluate the performances of automated protein function prediction methods. CAFA challenge is organised about every two years; however, it is a one-time challenge and it is not trivial to repeat the challenge with the same experimental settings afterwards. In addition, CAFA challenge does not provide any training dataset, machine learning models or different data splitting strategies (e.g., temporal, similarity-based and random split settings). To overcome these limitations, here, we propose an open-source Python package called ProFAB, Open Protein Functional Annotation Benchmark platform. The aim of ProFAB is to create a fair comparison platform for protein function prediction methods based on Gene Ontology (GO) (861,299 proteins annotated to 8360 function terms) and Enzyme Commission (EC) numbers (563,554 proteins annotated to 269 enzymatic functions). ProFAB supplies both positive and negative datasets separately for each function. To obtain numerical features from protein sequence data, various protein descriptors are provided: amino acid composition (AAC), pseudo amino acid composition (PAAC), sequence order coupling number (SOCNumber), conjoint triad (CTriad) and grouped amino acid composition (GAAC). ProFAB consists of 5 main functionalities: (i) Training dataset construction where several training/test/validation datasets are created using UniProtKB and UniRef databases, (ii) Data splitting where random, temporal and similaritybased splitting methods are provided, (iii) Standardization where several methods are used to scale the input features, (iv) Training and tuning of the classifiers which constitute support vector machine (SVM), random forest, k-nearest neighbor (KNN), decision trees, naive bayes, multilayer perceptron, gradient boosting, logistic regression and the ridge classifier (tuning of these classifiers is done automatically by the modules with the predefined parameters that are specific and modifiable for each machine learning algorithm), (v) Evaluation where several metrics are used to assess the predictive performance of trained models. To perform above functionalities, ProFAB uses Python modules and interfaces such as NumPY, scikit-learn, RDKit, pickle and tqdm. To convert protein sequences into numerical feature vectors, iLearn web tool [5] was used. With these implementations, we believe that ProFAB is useful for both computer scientists to find ready-to-use biological datasets, and wet-lab researchers to utilise ready-to-use machine learning algorithms to gain pre-knowledge about the functional view of proteins. ProFAB is available at https://github.com/Sametle06/ProFAB.git. To access the use case, please see: https://github.com/Sametle06/ProFAB/blob/master/test_file.ipynb.
                </p>
                </div>
              </p>

              <hr>

            </div>
          </div>
        </div>

        <div class="row">
          <div class="col-md-3 col-sm-3 col-xs-3 col-4"><p class="time">10:30 - 10:40</p></div>
          <div class="col-md-9 col-sm-9 col-xs-9 col-8"><p class="activity">Break</p></div>
        </div>

        <div class="row">
          <div class="col-md-3 col-sm-3 col-xs-3 col-4"><p class="time">10:40 - 11:10</p></div>
          <div class="col-md-5 col-sm-5 col-xs-5 col-6"><p class="invited-speaker">Tunca Doğan</p></div>
          <div class="col-md-4 col-sm-4 col-xs-4 col-2">
            <button type="button" class="btn-invited" data-toggle="collapse" data-target="#invited6">&nbsp;&#x2b07;&nbsp; Invited Talk</button>
          </div>
        </div>

        <div class="row">
          <div class="col-md-3 col-sm3 col-xs-3 col-4"></div>
          <div class="col-md-9 col-sm9 col-xs-9 col-8"><div id="invited6" class="collapse">
                <p class="text">
                <b>AI-centric approaches for integrating, associating, and analyzing large-scale and heterogeneous biomedical data</b>
                <br><br>
                <button class="btn-abstract" data-toggle="collapse" data-target="#collapsei6"><i class="fas fa-pencil-alt" aria-hidden="true"></i>Abstract</button>
                <div class="collapse" id="collapsei6">
                  <p class="text">
                    The recent availability of inexpensive technologies led to a surge of biological/biomedical data production and accumulation in public servers. These noisy, complex and large-scale data should be analyzed in order to understand mechanisms that constitute life and to develop new and effective treatments against prevalent diseases. A key concept in this endeavor is the prediction of unknown attributes and properties of biomolecules such as their molecular functions, physical interactions and etc., together with their relationships to high-level biomedical concepts such as systems and diseases. Lately, cutting-edge data-driven approaches are started to be applied to biological data to aid the development of novel and effective in silico solutions to problems in biomedicine. In this talk, I’ll summarize our efforts for integrating and representing heterogeneous data from different biological/biomedical data resources using graph-based technologies, together with the development and application of deep learning-based discriminative and generative computational methods for enriching the data by predicting the unknown properties and drug discovery centric ligand interactions of biomolecules.
                  </p>
                </div>
              </p>
              </div></div>
        </div>

        <div class="row">
          <div class="col-md-3 col-sm-3 col-xs-3 col-4"><p class="time">11:10 - 11:50</p></div>
          <div class="col-md-9 col-sm-9 col-xs-9 col-8">
            <button type="button" class="btn" data-toggle="collapse" data-target="#talk5">&nbsp;&#x2b07;&nbsp; Selected Talk Session - 5</button>
            <div id="talk5" class="collapse">

              <p class="text">

                <br>
                <b>Language Models can Learn Complex Functional Properties of Proteins</b>
                <br>
                <i>Serbulent Unsal, Heval Ataş, Muammer Albayrak, Kemal Turhan, Aybar Acar and Tunca Dogan</i>
                <br><br>
                <button class="btn-abstract" data-toggle="collapse" data-target="#collapses13"><i class="fas fa-pencil-alt" aria-hidden="true"></i>Abstract</button>
                <div class="collapse" id="collapses13">
                  <p class="text">
                    Proteins are essential macromolecules for life. To understand and manipulate biological mechanisms, functions of proteins should be understood, and this is possible through studying their relationship with the amino acid sequence and 3-D structure. So far, only a small percentage of proteins could be functionally characterized (currently ~0.5% according to UniProt) due to cost and time requirements of wet-lab-based procedures. Lately, protein function prediction (PFP), which can be defined as the annotation of proteins with functional definitions using statistical/computational methods, gains importance to explore the uncharacterized protein space and/or protein variants carrying function altering changes. Among many different algorithmic approaches proposed so far, machine learning (ML), especially deep learning (DL), techniques have become popular in PFP due to their high predictive performance. The input data used by these ML/DL methods are numerical feature vectors representing the protein (i.e., protein representations), and they are mostly generated from amino acid sequences of proteins which are readily available in databases (e.g., UniProt).
                    <br><br>
                    Early protein representation construction methods built upon evolutionary relationships, the composition of amino acids in the sequence, and/or the physicochemical features of amino acids, which are directly related to the biochemical function of the protein. These methods can be considered classical “model-driven” approaches. Recently, researchers started to utilize ML models to automatically learn protein representations from available protein data (e.g., protein sequence, protein-protein interactions, biomedical literature/texts), in the context of a “data-driven” approach called protein representation learning (PRL). Most of the novel PRL models are based on algorithms from the natural language processing (NLP) field (e.g., word2vec, doc2vec, LSTM/transformer-based BERT, XLNet, etc.), which are originally developed to model languages for automated and simultaneous translation, context-based text generation, etc. with elevated performances. In recent years, the number of PRL methods has multiplied and they are starting to be used in various areas from biomedicine to biotechnology. However, there is no comprehensive study and tool available to assess these representation methods in the context of modeling the functional properties of proteins, to help the researcher choose the suitable method for the task at hand.
                    <br><br>
                    In this study, we evaluated protein representation methods for the prediction of functional attributes of proteins and benchmarked these methods in 4 challenging tasks, namely: (i) Semantic similarity inference (we calculated pairwise semantic similarities between human proteins using their gene ontology annotations and compared them with representation vector similarities to observe the correlation in-between), (ii) Ontological protein function prediction (we built GO term categories based on term specificities and the sample sizes which reflects different levels of predictive difficulty and evaluated representation methods by training/validating ML models on these datasets), (iii) Drug target protein family classification (five major target families are selected and methods are evaluated in terms of classifying proteins to families via ML models), and (iv) Protein-protein binding affinity estimation (we used the SKEMPI dataset to evaluate methods in estimating protein-protein binding affinity changes upon mutations). We evaluated 23 protein representation methods in total, including both classical approaches and cutting-edge representation learning methods, to observe whether these novel approaches have advantages over classical ones, in terms of extracting high-level/complex properties of proteins that are hidden in their sequence. Finally, we provide an open-access tool, PROBE (Protein RepresentatiOn BEnchmark), where the user can assess new protein representation models over the above-mentioned benchmarking tasks with only a line of code.
                    <br><br>
                    The results of benchmarks showed that numerous DL-based PRL methods, especially large-scale protein language models, performed significantly better than classical representation methods on function and structural feature prediction-related tasks. Also, results indicated that the model architecture and training data types/sources are the two key factors affecting the performance. We also inspected possibilities of data leak from training to test, the cases where the tasks used during the pre-training are biologically related to the benchmark tasks (e.g., models that are constructed using Pfam protein family annotation data are good at predicting structural features since these two are directly related). Furthermore, we discussed current challenges in PRL such as differences between problems in the NLP domain and the ones in protein informatics, in the context of data structures and model interpretability. Finally, we discussed future applications of PRL in the fields of automated protein design and engineering. The details of methodology and results can be found in our preprint (https://doi.org/10.1101/2020.10.28.359828), which will be examined in detail and discussed further. PROBE/ProtBench is available at https://github.com/serbulent/TrainableRepresentationAnalysis.
                  </p>
                </div>
              </p>

              <hr>

              <p class="text">
                <b>DebiasedDTA: Model Debiasing to Boost Drug - Target Affinity Prediction</b>
                <br>
                <i>Rıza Özçelik, Alperen Bağ, Berk Atıl, Arzucan Özgür and Elif Özkırımlı</i>
                <br><br>
                <button class="btn-abstract" data-toggle="collapse" data-target="#collapses14"><i class="fas fa-pencil-alt" aria-hidden="true"></i>Abstract</button>
                <div class="collapse" id="collapses14">
                  <p class="text">
                    Prediction of drug-target affinity (DTA) in silico can significantly accelerate drug discovery process. Many in silico models rely on the drug-target interaction datasets, since they aim to learn the binding mechanisms between biomolecules through the information the datasets contain. However, the datasets on which the models rely also contain surface patterns, or dataset biases, that prevent models to generalize novel biomolecules [3, 2, 1]. Here we present DebiasedDTA, a model debiasing approach to boost the performance of DTA models on novel biomolecules. DebiasedDTA comprises a weak learner and a strong learner to identify and avoid dataset biases during training. With the fact that the non-complex models can memorize dataset biases easily, a weak learner is used to quantify the biases in the training samples. Once the bias is quantified, the strong learner avoids the biases by adjusting the training sample weights during training.
                    <br><br>
                    We experiment with 2 different weak learners to identify different bias sources: ID-DTA and BoW-DTA. ID-DTA is an identity-based weak learner that represents the biomolecules with one-hot encoding. On the other hand, BoW-DTA is a biomolecule-word-based approach that vectorizes the biomolecules with bag-of-words method, based on their chemical and protein words. Both weak learners concatenate chemical and protein representations to represent the interaction and use decision tree for regression.
                    <br><br>
                    We also experiment with 3 strong learners to observe the effect of debiasing with different strong learner architectures: DeepDTA, BPE-DTA, and LM-DTA. DeepDTA is a character-level-convolution based DTA model which was frequently used in the literature. DeepDTA uses SMILES strings of ligands and amino-acid sequences of proteinsfor biomolecule representation. We also design BPE-DTA, which uses the same model architecture as DeepDTA but segments sequences with Byte-Pair-Encoding instead of characters. Finally, we design LM-DTA that uses pre-trained biomolecule language model embeddings to represent the chemicals and proteins. Afterward, those representations are concatenated and fed into a 2 layered fully connected network.
                    <br><br>
                    We test DebiasedDTA on two datasets and evaluate the effect of debisasing on known and novel molecules separately. The results show that DebiasedDTA improves prediction performance on 44 of 48 experiments, suggesting that the proposed approach is useful in most scenarios. Both the known and novel biomolecules benefit from the performance boost and the boost is amplified when the test biomolecules are dissimilar to training set. The experiments also highlight that both identity and word based biases are prevalent in the datasets and each experimented strong learner can leverage the novel training scheme in DebiasedDTA, indicating that the proposed approach is generalizable across models. As such, we believe that DebiasedDTA will be an influential work for drug-target affinity prediction and will be used to debiase many future models.
                    <br><br>
                    References
                    <br>
                    [1] L. Chen, A. Cruz, S. Ramsey, C. J. Dickson, J. S. Duca, V. Hornak, D. R. Koes, and T. Kurtzman. Hidden bias in the dud-e dataset leads to misleading performance of deep learning in structure-based virtual screening. PloS one, 14(8):e0220113, 2019.
                    <br>
                    [2] J. Scantlebury, N. Brown, F. Von Delft, and C. M. Deane. Data set augmentation allows deep learning-based virtual screening to better generalize to unseen target classes and highlight important binding interactions. Journal of Chemical Information and Modeling, 60(8):3722–3730, 2020.
                    <br>
                    [3] J. Yang, C. Shen, and N. Huang. Predicting or pretending: artificial intelligence for protein-ligand interactions lack of sufficiently large and unbiased datasets. Frontiers in Pharmacology, 11:69, 2020.
                  </p>
                </div>
              <hr>

            </div>
          </div>
        </div>

        <div class="row">
          <div class="col-md-3 col-sm-3 col-xs-3 col-4"><p class="time">11:50 - 13:30</p></div>
          <div class="col-md-9 col-sm-9 col-xs-9 col-8">
            
            <button type="button" class="btn-poster" data-toggle="collapse" data-target="#poster2">&nbsp;&nbsp;&nbsp; &#x2b07; Poster Session</button>

            <div id="poster2" class="collapse">

              <p class="text">
                <b>(43) PhosProViz: A Web-based Tool to Generate and Interactively Explore Phosphoproteomics Networks</b>
                <br>
                <i>Irene Font Peradejordi, Shreya Chandrasekar, Berk Turhan, Selim Kalayci, Jeffrey Johnson and Zeynep Hülya Gümüş</i>
              </p>

              <hr>

              <p class="text">
                <b>(44) Metatranscriptome Analysis of Human Gut Microbiome by ASAIM Workflow</b>
                <br>
                <i>Ceyda Demirtaş and Seda Koldaş</i>
              </p>

              <hr>

              <p class="text">
                <b>(45) mirDisNet: A Novel Approach for Cancer Classification Using mir-Disease Associations</b>
                <br>
                <i>Amhar Jabeer, Burcu Bakir-Gungor and Malik Yousef</i>
              </p>

              <hr>

              <p class="text">
                <b>(46) Controversy Detection on Health-Related Tweets</b>
                <br>
                <i>Emine Ela Küçük, Selçuk Takır and Dilek Küçük</i>
              </p>

              <hr>

              <p class="text">
                <b>(47) Application of Machine Learning for The Identification of Novel Diagnostic Biomarkers for COVID-19 by Using Transcriptomic Data.</b> 
                <br>
                <i>Didem Ökmen, Athanasia Pavlopoulou and Eralp Doğu</i>
              </p>

              <hr>

              <p class="text">
                <b>(48) MicroBiomeNet: Machine Learning Analysis of Metagenomics Datasets: Colon Cancer Dataset</b> 
                <br>
                <i>Malik Yousef, Anas Nadifi, Amhar Jabeer and Burcu Bakir</i>
              </p>

              <hr>

              <p class="text">
                <b>(49) Protein Sequence Diversity Dynamics of Primate Erythroparvovirus 1</b> 
                <br>
                <i>Pendy Tok, Li Chuin Chong and Mohammad Asif Khan</i>
              </p>

              <hr>

              <p class="text">
                <b>(50) Protein Sequence Diversity of Human Respiratory Syncytial Virus</b> 
                <br>
                <i>Faruk Üstünel and M. Asif Khan</i>
              </p>

              <hr>

              <p class="text">
                <b>(51) Prediction of Regulatory Network Interactions with CNN Model using Human RNA-Seq Data</b> 
                <br>
                <i>Gülce Çelen and Alper Yılmaz</i>
              </p>

              <hr>

              <p class="text">
                <b>(52) Integrating Multi-Omics Data and Deep Learning for Discovering New Subtypes of Breast Cancer</b> 
                <br>
                <i>Hüseyin Uyar and Özgür Gümüş</i>
              </p>

              <hr>

              <p class="text">
                <b>(53) Survival Prediction of Sepsis Patients in an Intensive Care Unit</b> 
                <br>
                <i>Beste Kaysi and Ozgur Gumus</i>
              </p>

              <hr>

              <p class="text">
                <b>(54) Potential Implementation of Amino Acid Conjugates as Novel Micronutrient Fertilizers</b> 
                <br>
                <i>Emre Aksoy</i>
              </p>

              <hr>

              <p class="text">
                <b>(55) Functional Stratification of Small Molecule Drugs through Integrated Network Similarity</b> 
                <br>
                <i>Seyma Unsal Beyge and Nurcan Tuncbag</i>
              </p>

              <hr>

              <p class="text">
                <b>(56) Towards Integrative Mechanistic Models of Mammalian Cell Responses to Extracellular Perturbations: Growth Factors, Hormones, and Cytokines</b> 
                <br>
                <i>Cemal Erdem, Sean M. Gross, Laura M. Heiser and Marc R. Birtwistle</i>
              </p>

              <hr>

              <p class="text">
                <b>(57) Sequence Diversity of M Proteins of Influenza A (H7N9) Virus</b> 
                <br>
                <i>Gizem Yılmaz, Li Chuin Chong, Hasiba Karimi, Eyyüb Selim Ünlü, Muhammed Miran Öncel and Mohammad Asif Khan</i>
              </p>

              <hr>

              <p class="text">
                <b>(58) Explainable Artificial Intelligence Perspective to the Computational Drug Discovery Process</b> 
                <br>
                <i>Kevser Kübra Kırboğa and Ecir Uğur Küçüksille</i>
              </p>

              <hr>

              <p class="text">
                <b>(59) Classifying Antibiotic Resistance Mechanisms in Dihydrofolate Reductase by Tracking Dynamical Shifts in Hydrogen Bond Occupancies</b> 
                <br>
                <i>Ebru Çetin, Ali Rana Atilgan and Canan Atilgan</i>
              </p>

              <hr>

              <p class="text">
                <b>(61) Expression Profile Survey of Circular RNAs and Their Parent Genes in Context of Tissue Specificity</b> 
                <br>
                <i>Elif İrem KeleŞ and Alper Yılmaz</i>
              </p>

              <hr>

              <p class="text">
                <b>(62) Investigation of Radicals Present in Biological Systems by Molecular Modeling Methods</b>
                <br>
                <i>Buşra Baş and Cenk Selçuki</i>
              </p>

              <hr>

              <p class="text">
                <b>(63) Runs of Homozygosity Show That Human Inbreeding Has Decreased in Time through the Holocene</b> 
                <br>
                <i>Kanat Gürün, Francisco C. Ceballos, Ezgi Altınışık, Hasan Can Gemici, Cansu Karamurat, Dilek Koptekin, Kıvılcım Başak Vural, Igor Mapelli, Ekin Sağlıcan, Elif Sürer, Yılmaz Selim Erdal, Anders Götherström, Füsun Özer, Çiğdem Atakuman and Mehmet Somel</i>
              </p>

              <hr>

              <p class="text">
                <b>(64) Biomarker Prediction for Parkinson’s Disease by Transcriptome Mapping on a Genome-Scale Metabolic Model</b> 
                <br>
                <i>Ecehan Abdik and Tunahan Çakır</i>
              </p>

              <hr>

              <p class="text">
                <b>(65) Constraint-Based Modelling and Machine Learning Identifies Metabolic Alterations in the Substantia Nigra in Parkinson’s Disease</b> 
                <br>
                <i>Regan Odongo and Tunahan Çakır</i>
              </p>

              <hr>

              <p class="text">
                <b>(66) Reconstruction and Transcriptome-based Analysis of Rat Brain Specific Genome Scale Metabolic Network Model for Parkinson’s Disease</b> 
                <br>
                <i>Orhan Bellur and Tunahan Çakir</i>
              </p>

              <hr>

              <p class="text">
                <b>(67) Discovery of Latent Drivers from Double Mutations in Pan-Cancer Data Reveal their Clinical Impact</b> 
                <br>
                <i>Bengi Ruken Yavuz, Chung-Jung Tsai, Ruth Nussinov and Nurcan Tunçbağ</i>
              </p>

              <hr>

              <p class="text">
                <b>(68) ProFAB – Open Protein Functional Annotation Benchmark</b> 
                <br>
                <i>Ahmet Samet Özdilek, Ahmet Atakan, Tunca Doğan, Rengül Çetin-Atalay, Mehmet Volkan Atalay and Ahmet Süreyya Rifaioğlu</i>
              </p>

              <hr>

              <p class="text">
                <b>(69) Language Models can Learn Complex Functional Properties of Proteins</b> 
                <br>
                <i>Serbulent Unsal, Heval Ataş, Muammer Albayrak, Kemal Turhan, Aybar Acar and Tunca Dogan</i>
              </p>

              <hr>

              <p class="text">
                <b>(70) Consensus Clustering Analysis as a Sample Selection Method in Biomarker Discovery: Lung Cancer Case-Study</b> 
                <br>
                <i>Nehir Kızılilsoley and Emrah Nikerel</i>
              </p>

              <hr>

              <p class="text">
                <b>(71) Interaction Energy Analysis of Lidocaine and Papaverine with the Drug Carrier Pectin</b> 
                <br>
                <i>Nesrin Işıl Yaşar, Tuğçe İnan, Ayşe Özge Kürkçüoğlu levitas and Fethiye Aylin Sungur</i>
              </p>

              <hr>

              <p class="text">
                <b>(72) DebiasedDTA: Model Debiasing to Boost Drug - Target Affinity Prediction</b> 
                <br>
                <i>Rıza Özçelik, Alperen Bağ, Berk Atıl, Arzucan Özgür and Elif Özkırımlı</i>
              </p>

              <hr>

              <p class="text">
                <b>(73) Methylation Deviation as a Marker of Intratumor Heterogeneity and Cancer Progression</b> 
                <br>
                <i>Ersin Onur Erdoğan, Ömer Çinal and Mehmet Baysan</i>
              </p>

              <hr>

              <p class="text">
                <b>(74) Predicting the Impact of Cancer Somatic Mutations on Protein-Protein Interactions</b> 
                <br>
                <i>Ibrahim Berber, Cesim Erten and Hilal Kazan</i>
              </p>

              <hr>

              <p class="text">
                <b>(75) Drug-Target Interaction Prediction Using Transfer Learning</b> 
                <br>
                <i>Alperen Dalkiran, Ahmet Süreyya Rifaioğlu, Aybar Can Acar, Tunca Dogan, Rengul Atalay and Volkan Atalay</i>
              </p>

              <hr>

              <p class="text">
                <b>(76) Prediction of Resistance to Drugs in Triple Negative Breast Cancer Based on Gene Expression Levels</b> 
                <br>
                <i>Bengisu Karaköse, Berk Gürdamar and Osman Uğur Sezerman</i>
              </p>

              <hr>

              <p class="text">
                <b>(77) Predicting Oral Health Using Machine Learning</b> 
                <br>
                <i>Emrah Kırdök and Andres Aravena</i>
              </p>

              <hr>

              <p class="text">
                <b>(78) Archaeogenetic Analysis of Neolithic Sheep from Anatolia</b> 
                <br>
                <i>Erinç Yurtman, Onur Özer, Eren Yüncü, Nihan Dilşad Dağtaş, Dilek Koptekin, Yasin Gökhan Çakan, Mustafa Özkan, Ali Akbaba, Damla Kaptan, Gözde Atağ, Kıvılcım Başak Vural, Can Yumni Gündem, Louise Martin, Gülşah Merve Kılınç, Ayshin Ghalichi, Sinan Can Açan, Reyhan Yaka, Ekin Sağlıcan, Vandela Kempe Lagerholm, Maja Krzewinska, Torsten Günther, Pedro Morell Miranda, Evangelia Piskin, Müge Şevketoğlu, C. Can Bilgin, Çiğdem Atakuman, Yılmaz Selim Erdal, Elif Sürer, N. Ezgi Altınışık, Johannes Lenstra, Sevgi Yorulmaz, Mohammad Foad Abazari, Javad Hoseinzadeh, Douglas Baird, Erhan Bıçakçı, Özlem Çevik, Fokke Gerritsen, Rana Özbal, Anders Götherström, Mehmet Somel, İnci Togan and Füsun Özer</i>
              </p>

              <hr>

              <p class="text">
                <b>(79) Potential Inhibitor Identification for Deoxyhypusine Synthase</b> 
                <br>
                <i>Ayşenur Öztürk and Fethiye Aylin Sungur</i>
              </p>

              <hr>

              <p class="text">
                <b>(80) Inter-Tissue Convergence of Gene Expression and Loss of Cellular Identity During Ageing</b> 
                <br>
                <i>Hamit İzgi, Dingding Han, Ulas Isildak, Shuyun Huang, Ece Kocabiyik, Philipp Khaitovich, Mehmet Somel and Handan Melike Dönertaş</i>
              </p>

              <hr>

              <p class="text">
                <b>(81) Ensemble Learning Approach for Computational Drug Repurposing</b> 
                <br>
                <i>Ismail Denizli, Oğuzhan Şahİn, Özgür DoĞan, Tuğba Süzek and Baris Süzek</i>
              </p>

              <hr>

              <p class="text">
                <b>(82) Extraction of Herb-Drug Interactions</b> 
                <br>
                <i>Erkan Yaşar, Remzi Çelebi and Özgür Gümüş</i>
              </p>

              <hr>

              <p class="text">
                <b>(83) Identification of Autophagy-Related miRNA–mRNA Regulatory Network in Calorie-Restricted Mouse Brain</b> 
                <br>
                <i>Atakan Ayden, Elif Yılmaz, Bilge G. Tuna, Ayşegül Kuskucu, Ömer F. Bayrak, Andres Aravena and Soner Doğan</i>
              </p>

              <hr>

            </div>
          </div>
        </div>

        <div class="row">
          <div class="col-md-3 col-sm-3 col-xs-3 col-4"><p class="time">13:30 - 14:00</p></div>
          <div class="col-md-5 col-sm-5 col-xs-5 col-6"><p class="invited-speaker">Maria Secrier</p></div>
          <div class="col-md-4 col-sm-4 col-xs-4 col-2">
            <button type="button" class="btn-invited" data-toggle="collapse" data-target="#invited7">&nbsp;&#x2b07;&nbsp; Invited Talk</button>
          </div>
        </div>

        <div class="row">
          <div class="col-md-3 col-sm3 col-xs-3 col-4"></div>
          <div class="col-md-9 col-sm9 col-xs-9 col-8"><div id="invited7" class="collapse">
                <p class="text">
                <b>Genomic triggers and evolutionary context of cancer dormancy</b>
                <br><br>
                <button class="btn-abstract" data-toggle="collapse" data-target="#collapsei7"><i class="fas fa-pencil-alt" aria-hidden="true"></i>Abstract</button>
                <div class="collapse" id="collapsei7">
                  <p class="text">
                    Tumour dormancy, a state in which cancer cells are reversibly arrested in the cell cycle, is frequently reported as a contributing factor of resistance to chemotherapy and other treatments that target cycling cells. However, despite its crucial role in cancer progression, dormancy is still poorly characterised and the molecular changes enabling the transition to this state remain largely unknown. Cellular stress can drive cells into dormancy as a mechanism to avoid further DNA damage and maintain genomic stability. How these shifts occur due to various mutational processes and how they impact cancer progression has not been elucidated. I will present an integrated computational framework for evaluating tumour dormancy and its genomic triggers across a variety of cancers. We show that dormancy preferentially emerges in the context of more stable, less mutated genomes which maintain TP53 integrity and lack the hallmarks of DNA damage repair deficiency. Using an ensemble elastic net regression model, we uncover several novel genomic dependencies of this process that could be exploited to maintain or promote exit from this state. We also use single cell data to demonstrate quiescence is a key resistance mechanism to a wide range of cancer therapies. Finally, we demonstrate broad reorganisation of the tumour tissue in the context of dormancy, which could inform therapeutic strategies.
                  </p>
                </div>
              </p>
              </div></div>
        </div>

        <div class="row">
          <div class="col-md-3 col-sm-3 col-xs-3 col-4"><p class="time">14:00 - 14:30</p></div>
          <div class="col-md-5 col-sm-5 col-xs-5 col-6"><p class="invited-speaker">Rayan Chikhi</p></div>
          <div class="col-md-4 col-sm-4 col-xs-4 col-2">
            <button type="button" class="btn-invited" data-toggle="collapse" data-target="#invited8">&nbsp;&#x2b07;&nbsp; Invited Talk</button>
          </div>
        </div>

        <div class="row">
          <div class="col-md-3 col-sm3 col-xs-3 col-4"></div>
          <div class="col-md-9 col-sm9 col-xs-9 col-8"><div id="invited8" class="collapse">
                <p class="text">
                <b>A tale of optimizing the space usage of de Bruijn graphs</b>
                <br><br>
                <button class="btn-abstract" data-toggle="collapse" data-target="#collapsei8"><i class="fas fa-pencil-alt" aria-hidden="true"></i>Abstract</button>
                <div class="collapse" id="collapsei8">
                  <p class="text">
                    In the last decade in bioinformatics, many computational works have studied a graph data structure used to represent genomic data, the de Bruijn graph. It is closely tied to the problem of genome assembly, i.e. the reconstruction of an organism’s chromosomes using a large collection of overlapping short fragments. We start by highlighting this connection, noting that assembling genomes is a computationally intensive task, and then focus our attention on the various techniques developed to reduce the space taken by de Bruijn graph data structures. This talk is a retrospective aimed to be accessible without prior knowledge of this area.
                  </p>
                </div>
              </p>
              </div></div>
        </div>

        <div class="row">
          <div class="col-md-3 col-sm-3 col-xs-3 col-4"><p class="time">14:30 - 14:45</p></div>
          <div class="col-md-9 col-sm-9 col-xs-9 col-8"><p class="activity">Compecta</p></div>
        </div>
        
        <div class="row">
          <div class="col-md-3 col-sm-3 col-xs-3 col-4"><p class="time">14:45 - 15:00</p></div>
          <div class="col-md-9 col-sm-9 col-xs-9 col-8"><p class="activity">Break</p></div>
        </div>

        <div class="row">
          <div class="col-md-3 col-sm-3 col-xs-3 col-4"><p class="time">15:00 - 16:00</p></div>
          <div class="col-md-5 col-sm-5 col-xs-5 col-6"><p class="keynote-speaker">Ivet Bahar</p></div>
          <div class="col-md-4 col-sm-4 col-xs-4 col-2">
            <button type="button" class="btn-keynote" data-toggle="collapse" data-target="#keynote2">&nbsp;&#x2b07;&nbsp; Keynote Talk</button>
          </div>
        </div>

        <div class="row">
          <div class="col-md-3 col-sm3 col-xs-3 col-4"></div>
          <div class="col-md-9 col-sm9 col-xs-9 col-8"><div id="keynote2" class="collapse">
                <p class="text">
                <b>Network models in biology: Molecular machinery, chromosomal dynamics, and pathogenicity of missense variants</b>
                <br><br>
                <button class="btn-abstract" data-toggle="collapse" data-target="#collapsek2"><i class="fas fa-pencil-alt" aria-hidden="true"></i>Abstract</button>
                <div class="collapse" id="collapsek2">
                  <p class="text">
                    Network models have proven in recent years to assist in improving our understanding of the coupled dynamics of biomolecules, from individual proteins to supramolecular systems, and even the entire chromatin, in recent years. Among network models that have been developed for biological applications, elastic network models (ENMs) found wide usage in molecular biology 1 . The global motions predicted by ENMs have proven in numerous applications in the last two decades to provide a good description of molecular machinery and allosteric behavior. Application to supramolecular structures, including cryo-EM structures, has been a major utility of these models. More recently, ENMs proved useful to exploring chromosomal dynamics, using data from Hi-C experiments to reconstruct in silico the connectivity of the chromatin and provide a physical basis for gene regulation transcription and cell type differentiation 2 . Finally, machine learning algorithms that incorporate ENM predictions provide an improved assessment of the effect of mutations on function, compared to those based on sequence and structure exclusively 3,4 . These recent developments and future biomedical and pharmacological applications will be discussed.
                    <br><br>
                    References
                    <br>[1] Krieger JM, Doruker P, Scott AL, Perahia D, Bahar I. (2020) Towards Gaining Sight of Multiscale Events: Utilizing Network Models and Normal Modes in Hybrid Methods. Curr Opin Struct Biol 64:34-41.
                    <br>[2] Zhang S, Chen F, Bahar I (2020) Differences in the Intrinsic Spatial Dynamics of the Chromatin Contribute to Cell Differentiation. Nucleic Acids Res. 48, 1131-1145.
                    <br>[3] Ponzoni L, Bahar I. (2018) Structural dynamics is a determinant of the functional significance of missense variants. Proc Natl Acad Sci USA 115: 4164-4169
                    <br>[4] Ponzoni L, Penaherrera DA, Oltvai ZN, and Bahar I (2020) Rhapsody: Predicting the pathogenicity of human missense variants. Bioinformatics 36:3084-3092.
                    <br>
                  </p>
                </div>
              </p>
              </div></div>
        </div>

        <div class="row">
          <div class="col-md-3 col-sm-3 col-xs-3 col-4"><p class="time">16:00 - 16:40</p></div>
          <div class="col-md-9 col-sm-9 col-xs-9 col-8">
            <button type="button" class="btn" data-toggle="collapse" data-target="#talk6">&nbsp;&#x2b07;&nbsp; Selected Talk Session - 6</button>
            <div id="talk6" class="collapse">

              <p class="text">

                <br>
                <b>Inter-tissue convergence of gene expression and loss of cellular identity during ageing</b>
                <br>
                <i>Hamit İzgi, Dingding Han, Ulas Isildak, Shuyun Huang, Ece Kocabiyik, Philipp Khaitovich, Mehmet Somel and Handan Melike Dönertaş</i>
                <br><br>
                <button class="btn-abstract" data-toggle="collapse" data-target="#collapses15"><i class="fas fa-pencil-alt" aria-hidden="true"></i>Abstract</button>
                <div class="collapse" id="collapses15">
                  <p class="text">Studying gene expression changes during ageing gives insight to identifying age-related molecular and cellular processes. Recent molecular studies including these two periods have reported a reversal of the ageing transcriptome towards pre-adult levels in primate brain and mouse liver and kidney. Several major questions remain to be answered. Prevalence of reversal phenotypes across tissues is yet unclear and is not studied extensively, as most research has been conducted in the brain. To address this, we generated RNA-seq transcriptomes of 16 mice from four tissues; cortex, lung, liver and muscle, covering development and ageing intervals. First, we revealed that approximately %50 of the genes showed expression reversals in each tissue, although these proportions are not significant in permutation tests, suggesting that the expression trajectories of the genes do not necessarily continue linearly into the ageing period. Functional analysis of the genesets showing reversal patterns identified pathways related to development, metabolism and inflammation. Next, we asked whether different tissues show similarities in their reversal patterns. We found no significant overlap in reversal genes among tissues suggesting that expression reversals might be tissue-specific. In concordance with the tissue-specific reversals, we showed that during development, tissues become more distinct (diverge) and interestingly during ageing, tissues become more similar (converge) in their gene expression levels. We confirmed this observation using two other independent RNA-seq (human) and microarray datasets (mouse). Moreover, divergence-convergence pattern is enriched among tissue-specific genes which show either decreased expression in native tissue or gaining expression in non-native tissue during ageing. Further, using publicly available single-cell transcriptome data, we showed that divergence-convergence pattern is driven both by alterations in cell type proportions and also by cell-autonomous expression changes. This supports our previous hypothesis that loss of cellular identity during ageing might be a general phenomenon in mammalian ageing.
                  </p>
                </div>
              </p>

              <hr>

              <p class="text">
                <b>Comparison and assessment of speed and accuracy of AutoDock Vina and AutoDock CrankPep for short peptide docking</b>
                <br>
                <i>Sefer Baday and Numan Nusret Usta</i>
                <br><br>
                <button class="btn-abstract" data-toggle="collapse" data-target="#collapses16"><i class="fas fa-pencil-alt" aria-hidden="true"></i>Abstract</button>
                <div class="collapse" id="collapses16">
                  <p class="text">
                    Protein-peptide interactions (PPI) are one of most interesting area of molecular biology due to their importance in biological processes. In computational biology, molecular docking is commonly used technique for investigating the PPI. However, due to the flexible structure of peptides, docking of peptides to proteins are very challenging due to high conformational degrees of freedom. Various molecular docking tools are available with different search and scoring algorithms for different docking needs such as protein-protein, protein-ligand, nucleic acid-ligand and etc. For investigating the effectiveness of molecular docking tools, some benchmark studies were done by different research groups. But, especially for protein-peptide docking, these studies could seem inadequate in terms of number of studied complexes and peptide sequence length diversity of complexes. By considering these circumstances, we choose PepBDB which is a comprehensive structural database of biological protein-peptide complexes. PepBDB has more than 13000 complexes that their peptide lengths ranged from 1 to 50. In this study, we worked with two different tools of most common free docking softwares AutoDock CrankPep (ADCP) and AutoDock Vina. Accuracy of results are determined with RMSD. We performed local docking for all complexes and evaluated the top pose of each of them. We analyzed docking results of the peptides by their sequence length. Docking results of shorter peptides are better compared to longer ones for both of the docking tools. Also, time consumption of longer peptides is much more in terms of peptide per minute. Success rate of ADCP is noticeably better than Vina for shorter peptides. However, the gap between these programs is decreasing as peptide length increases. We also determined the optimal parameters for both programs by the sequence length.
                  </p>
                </div>
              </p>

              <hr>

            </div>
          </div>
        </div>

        <div class="row">
          <div class="col-md-3 col-sm-3 col-xs-3 col-4"><p class="time">16:40 - 17:00</p></div>
          <div class="col-md-9 col-sm-9 col-xs-9 col-8"><p class="activity">Break</p></div>
        </div>

        <div class="row">
          <div class="col-md-3 col-sm-3 col-xs-3 col-4"><p class="time">17:00 - 18:00</p></div>
          <div class="col-md-5 col-sm-5 col-xs-5 col-6"><p class="keynote-speaker">Mehmet Koyutürk</p></div>
          <div class="col-md-4 col-sm-4 col-xs-4 col-2">
            <button type="button" class="btn-keynote" data-toggle="collapse" data-target="#keynote3">&nbsp;&#x2b07;&nbsp; Keynote Talk</button>
          </div>
        </div>
        
        <div class="row">
          <div class="col-md-3 col-sm3 col-xs-3 col-4"></div>
          <div class="col-md-9 col-sm9 col-xs-9 col-8"><div id="keynote3" class="collapse">
            <p class="text">
                <b>Overcoming Bias in Computational Characterization of Cellular Signaling</b>
                <br><br>
                <button class="btn-abstract" data-toggle="collapse" data-target="#collapsek3"><i class="fas fa-pencil-alt" aria-hidden="true"></i>Abstract</button>
                <div class="collapse" id="collapsek3">
                  <p class="text"> Protein phosphorylation is a key regulator of protein function in signal transduction pathways. Kinases are the enzymes that catalyze the phosphorylation of other proteins in a target-specific manner. The dysregulation of phosphorylation is associated with many diseases, including cancers, neuro-degenerative disorders, and auto-immune diseases. Consequently, characterization of kinase activity in the context of these diseases is essential for the development of effective treatments.Indeed, in the last decade, kinase inhibitors have become central to the treatment of a broad range of cancers.  Although technological advances enable the identification of phosphorylated sites for thousands of proteins, most of the phosphoproteome is still in the dark: more than 95% of the reported phosphorylation sites in humans have no known kinases. The incompleteness of kinase annotations and the limitations of data acquisition techniques give rise to two important computational problems in the study of cellular signaling: 1) How can we identify the kinases that target a given phosphorylation site? 2) How can we identify the kinases with altered activity based on the changes in the phosphorylation levels of their targets? In this talk, we demonstrate that the computational tools that are developed to address these fundamental problems have a common flaw that poses limitations on the expansion of knowledge: Both benchmarking data and algorithms are biased toward well-studied kinases. We then describe the algorithms we develop to overcome these hurdles by integrating a broad range of functional data.  Finally, we present comprehensive results that aim to systematically assess the robustness of algorithms with respect to missing data. Our results show that network algorithms can significantly enhance the utility of phospho-proteomic data and the ability of computational tools in generating new knowledge that pertains to cellular signaling. </p>
                </div>
              </p>
            </div>
          </div>
        </div>

        <div class="row">
          <div class="col-md-3 col-sm-3 col-xs-3 col-4"><p class="time">18:00 - 18:15</p></div>
            <div class="col-md-9 col-sm-9 col-xs-9 col-8">
            <p class="activity"><b>Closing Remarks</b></p>
          </div>
        </div>

      </div>
    </div> <!-- End row -->


<!--
      <div class="row float-right">*Time Zone: GMT+3</div>-->
      

  </div> <!-- End container -->
</section> <!-- End section -->
<!-- End Program
    ==================================== -->
    

<!-- begin Footer Area
    ========================================== -->
<footer id="footer" class="bg-one">
  <div class="top-footer">
    <p>Any questions? Email organizers at <b>recomb.conference@gmail.com</b></p>
  </div>
  <div class="footer-bottom">
    <h5>Copyright 2024. All rights reserved.</h5>
    <h6>Design and Developed by Themefisher</h6>
  </div>
</footer> <!-- end footer -->
<!-- end Footer Area
    ========================================== -->

<!-- Essential Scripts
    =====================================-->
    <!-- Main jQuery -->
    <script src="plugins/jquery/jquery.min.js"></script>
    <!-- Google Map -->
    <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyBu5nZKbeK-WHQ70oqOWo-_4VmwOwKP9YQ"></script>
    <script  src="plugins/google-map/gmap.js"></script>

    <!-- Form Validation -->
    <script src="plugins/form-validation/jquery.form.js"></script> 
    <script src="plugins/form-validation/jquery.validate.min.js"></script>
    
    <!-- Bootstrap4 -->
    <script src="plugins/bootstrap/js/bootstrap.min.js"></script>
    <!-- Parallax -->
    <script src="plugins/parallax/jquery.parallax-1.1.3.js"></script>
    <!-- lightbox -->
    <script src="plugins/lightbox2/dist/js/lightbox.min.js"></script>
    <!-- Owl Carousel -->
    <script src="plugins/slick/slick.min.js"></script>
    <!-- filter -->
    <script src="plugins/filterizr/jquery.filterizr.min.js"></script>
    <!-- Smooth Scroll js -->
    <script src="plugins/smooth-scroll/smooth-scroll.min.js"></script>
    
    <!-- Custom js -->
    <script src="js/script.js"></script>

  </body>
  </html>
